---
title: HARMONY 2023

# Is this the primary user of the site?
superuser: false

conf_date: 24-27, April 2023

conf_location: Seattle, WA

important_dates:
- name: Breakout submission deadline (for initial scheduling)
  date: April 2, 2023
- name: Lightning talk and poster submission deadline
  date: April 2, 2023
- name: Notification of acceptance of breakouts and abstracts
  date: April 7, 2023
- name: HARMONY Workshop
  date: April 24 to 27, 2023

important_links:
- name: Agenda
  link: https://combine-org.github.io/events/
- name: Registration
  link: https://forms.gle/Nqec2JdyRj12HobK9
- name: Breakout submission
  link: https://forms.gle/rYZhqbekKLs1RY3B7
- name: Lightning talk and poster submission
  link: https://forms.gle/hcpWJrKCtFQDuPwR6

information:
- name: Registration
  description: Registration for the meeting is open and is free. Please register at the link above as soon as possible. This will help us plan the schedule and match your interests to the timing of the breakouts, etc. Note, only registered attendees will be sent information related to video conferencing links, etc.
  link: https://forms.gle/Nqec2JdyRj12HobK9
- name: Call for Breakout Sessions
  description: All attendees can suggest breakout sessions for hacking and/or detailed discussions of certain aspects of one or several of the COMBINE standard(s), metadata and semantic annotations (format-specific or overarching), application and implementations of the COMBINE standards, or any other topic relevant for the COMBINE community. The topics for those breakout sessions, and the time slots which would suit their communities can be submitted via the link above. Note, breakout session organisers will be responsible for creating and hosting their own online sessions, if required.
  link: https://forms.gle/rYZhqbekKLs1RY3B7
- name: Call for Lightning Talks and Posters
  description: Requests for a lightning talk (3 min max.) and/or poster can be submitted via the link above. Please use several forms if you want to submit abstracts on different topics. The submission deadline is outlined above. Talks will take place during the community session and posters will be displayed throughout the meeting.
  link: https://forms.gle/hcpWJrKCtFQDuPwR6

topics:
- Data exchange, pipelines and model standards for systems and synthetic biology
- Visualization and graphical notation standards for systems and synthetic biology
- Standards for sharing and analysing biological pathway data
- Standards for computational biological models and modelling support
- Metadata description and model annotation in COMBINE standard formats
- Implementation of COMBINE standards in tools, databases and other resources
- Integrated model and data management for systems and synthetic biology
- Standardization of Artificial Intelligence approaches in biological modelling
- Emerging standardization needs and multicellular modeling
- Community aspects of COMBINE

# Organizational groups that you belong to (for People widget)
#   Set this to `[]` or comment out if you are not using People widget.
user_groups:
- Past Events

---
The Computational Modeling in Biology Network (COMBINE) is an initiative to coordinate the development of the various community standards and formats in systems biology, synthetic biology and related fields. HARMONY is a codefest-type meeting, with a focus on development of the standards, interoperability and infrastructure. There are generally not many general discussions or oral presentations during HARMONY; instead, the time is devoted to allowing hands-on hacking and interaction between people focused on practical development of software and standards.

HARMONY 2023 was hosted by Herbert Sauro and the Center for Reproducible Biomedical Modeling at the University of Washington, Seattle, WA. It was organized primarily as an in-person meeting, with individual breakout sessions responsible for enabling remote participation as needed.

Most of each day was scheduled by the communities as breakouts. In addition, we had some time each day for community discussion and wrap-ups of breakouts and advertisements for following breakouts.

<h3> Schedule </h3>

Here's the [full (editable) schedule](https://docs.google.com/spreadsheets/d/1Jjs22pMGjJZfeT7tp8btoJxQLXnCdWysEIKn26U72_4/edit#gid=0).  Note that many events are scheduled somewhat spontaneously at these events; keep an eye out here or on the COMBINE discord for last-minute changes and additions.

<h3>Workshop Location Details</h3>

HARMONY 2023 takes place at the University of Washington (UW, and commonly called "U-Dub") in Seattle, Washington. HARMONY 2023 will take place in the William H. Foege Bioengineering building, at [3720 15th Ave NE, Seattle, WA 98195](https://goo.gl/maps/rEGPZXcLkiHBcwBa7).  The building is locked during the day, but will be open the half-hour before the meeting begins.  If you arrive later than that, call the department reception at 206-685-2000, or email any of the organizers, or ping us on the [COMBINE slack channel](combine-workspace.slack.com).

<img src="/images/harmony2023/foege and map.png" width="95%" alt="Foege building and map">

The doors will be locked, so please knock (there's lots of glass), call the Department front desk at (206) 685-2000 or email lpsmith@uw or bioeasst@uw.edu or ping us on the COMBINE Slack!


<h3>Arrival and Transportation</h3>

From the Seatac Airport, take the [Link Light Rail](https://www.soundtransit.org/ride-with-us/stations/link-light-rail-stations) north from the airport, and exit at the University of Washington station (NOT Univeristy *Street* Station; that's a downtown stop).  It's a relatively short walk [from there to the Bioengineering building](https://goo.gl/maps/82vspAtwTpG439xN7).  The nearest hotel is [slightly further](https://goo.gl/maps/6TMqczLss91J3TZL7); a little over one mile, so you might want to bus/taxi depending on how much luggage you have.  If you get an [ORCA card](https://www.soundtransit.org/ride-with-us/how-to-pay), you can use that on any of the buses or other light rail trips you might take during your stay.  There's also a ['Transit GO Ticket'](https://kingcounty.gov/en/depts/metro/fares-and-payment/ways-to-pay/transit-go-ticket) app you should be able to use from your phone (untested by me, but it seems reasonable?).

<h3>Hotels</h3>

The closest hotel to the Bioengineering department is the [Univeristy Inn](https://www.staypineapple.com/university-inn-seattle-wa) at [4140 Roosevelt Way NE, Seattle, WA 98105](https://goo.gl/maps/eZ7bJMLNi5hBWr2H6); it's about a [half mile walk](https://goo.gl/maps/bdWK28hoyLREGFff8) between there and the venue.  About a block further (and inexplicably slightly more expensive) is the [Watertown Hotel](https://www.staypineapple.com/watertown-hotel-seattle-wa), at [4242 Roosevelt Way NE, Seattle, WA 98105](https://goo.gl/maps/3k6jrAKmF9JocTo66).  Any hotel anywhere along the [Link Light Rail](https://www.soundtransit.org/ride-with-us/stations/link-light-rail-stations) line will be easy to get to/from as well, as the Univeristy of Washington station is just a half mile away from the venue.

<h3> Attendees </h3>

The following is the list of known attendees as of 24/04/2023 23:58:54 UTC (72 total registered, 63 listed below)

| Name | Affiliation | Attendance |
|:---|:---|:---|
| Eran Agmon | UConn Health | In person <br/> Multicellular modeling, SBML, SED-ML <br/>BioSimulators: https://biosimulators.org, Vivarium: https://vivarium-collective.github.io |
| Azka Ahmed | UW Madison | Remotely (virtually) <br/> BioPAX, CellML, COMBINE Archive, Multicellular modeling  |
| Daniel Ajuzie | Wisconsin Institute for Discovery | Remotely (virtually) <br/> BioPAX, CellML, COMBINE Archive, Multicellular modeling, SBML, SED-ML <br/>Our lab develops models at multiple scales ranging from gene network, metabolic network, signal transduction to multicellular models. We model individual cells (bacteria and mammalian) and interactions between cells (microbial and host-microbe). |
| Steve Andrews | UW | In person <br/>  <br/>Spatial modeling, including particularly with the Smoldyn package. |
| Mihail Anton | NBIS, SciLifeLab, Chalmers University of Technology | Remotely (virtually) <br/> COMBINE Archive, OMEX Metadata, SBGN, SBML <br/>standard-GEM |
| Bryan Bartley | Raytheon BBN | In person <br/> SBOL and SBOL Visual  |
| Jacob Beal | Raytheon BBN | In person <br/> SBOL and SBOL Visual  |
| Frank Bergmann | Heidelberg University | In person <br/> COMBINE Archive, SBGN, SBML, SED-ML <br/>COPASI (copasi.org), basico (basico.rtfd.io), libsbml, libsedml, libcombine |
| Lutz Brusch | Technische Universität Dresden, Germany | Remotely (virtually) <br/> Multicellular modeling, SBML <br/>COMBINE Archive and OMEX, PEtab, MorpheusML, SBML-Spatial, FAIRSPACE, https://www.MultiCellML.org  |
| Lukas Buecherl | University of Colorado Boulder | In person <br/> COMBINE Archive, SBML, SBOL and SBOL Visual, SED-ML <br/>Genetic Design Automation |
| Kevin Chen | University of Washington | In person <br/> BioPAX, CellML, COMBINE Archive, NeuroML, OMEX Metadata, SBML, SED-ML  |
| Rafael Costa | FCT-NOVA, NOVA School of Science and Technology | Remotely (virtually) <br/> COMBINE Archive, SBML  |
| Alan Garny | University of Auckland | In person <br/> CellML, COMBINE Archive, Multicellular modeling, OMEX Metadata, SED-ML <br/>libOpenCOR and OpenCOR |
| Tom Gebhardt | University of Greifswald | Remotely (virtually) <br/> CellML, COMBINE Archive, SBGN, SBML  |
| John Gennari | U of Washington | In person <br/> BioPAX, CellML, OMEX Metadata, SBML, SBOL and SBOL Visual, SED-ML  |
| James Glazier | Indiana University | Remotely (virtually) <br/> COMBINE Archive, Multicellular modeling, SBML <br/>Replication and annotation of multicellular models |
| Issa Gnasse | Pasteur Institute of Dakar | In person <br/> Multicellular modeling, OMEX Metadata <br/>Application of Machine learning to find early biomarker in cervical cancer. |
| Misirli Goksel | Keele University | Remotely (virtually) <br/> COMBINE Archive, OMEX Metadata, SBML, SBOL and SBOL Visual  |
| Jenn Hadlock | Insitute for Systems Biology | Remotely (virtually) <br/> BioPAX, OMEX Metadata, SBML <br/>NCATS Biomedical Translator |
| Joseph Hellerstein | eScience Institute, University of Washington | In person <br/> COMBINE Archive, SBML <br/>VSCode-Antimony smart editor, AMAS (Automated Model Annotation System) |
| Ron Henkel | Medical Informatics Laboratory, Institute for Community Medicine, University Medicine Greifswald | In person <br/> CellML, COMBINE Archive, SBGN, SBML <br/>FAIR |
| Adel Heydarabadipour | University of Washington | In person <br/> Multicellular modeling, SBGN, SBML, SBOL and SBOL Visual  |
| Yan-kay Ho | University of Cambridge | Remotely (virtually) <br/> SBML, SBOL and SBOL Visual <br/>https://openbioeconomy.org/research_categories/toolkits/ https://github.com/openplant/openplant_automation_protocols |
| Diego Jahn | Center for Information Services and High Performance Computing (ZIH), TU Dresden, Germany | Remotely (virtually) <br/> Multicellular modeling, MultiCellML, MorpheusML <br/>Morpheus (https://morpheus.gitlab.io/), MorpheusML Model Repository (https://morpheus.gitlab.io/model/) |
| Bart Jardine | University of Washington | In person <br/> COMBINE Archive, Multicellular modeling, OMEX Metadata, SBML, SED-ML <br/>Chemical network modeling software and support for SBML |
| Paul jonas Jost | University of Bonn | Remotely (virtually) <br/> COMBINE Archive, SBGN, SBML <br/>pyPESTO, petab |
| Jonathan Karr | Formic Labs | Remotely (virtually) <br/> COMBINE Archive, OMEX Metadata, SBML, SED-ML <br/>BioSimulators, BioSimulations |
| Sarah Keating | UCL | Remotely (virtually) <br/> SBML  |
| Carlos Lopez | Altos Laboratories | Remotely (virtually) <br/> Multicellular modeling, SBML, models as computer programs <br/>PySB, PyDREAM, PyViPR, all things multiscale modeling |
| Augustin Luna | Harvard Medical School | Remotely (virtually) <br/> BioPAX, OMEX Metadata, SBGN, SBOL and SBOL Visual <br/>Pathway Commons (pathwaycommons.org); SBGN (sbgn.org); libsbgn (github.com/sbgn/libsbgn); stonpy (github.com/adrienrougny/stonpy) |
| Karin Lundengård | Auckland Bioengineering Institute | Remotely (virtually) <br/>   |
| Harvey Mannering | UCL | In person <br/> NeuroML, SBML, SED-ML <br/>Tools and standards for single-cell modeling in neuroscience.  Currently involved in the ongoing development of libSBML |
| Elebeoba May | University of Wisconsin-Madison | Remotely (virtually) <br/> BioPAX, CellML, COMBINE Archive, Multicellular modeling, SBML, SED-ML <br/>My lab develops models at multiple scales ranging from gene network, metabolic network, signal transduction to multicellular models.  We model individual cells (bacteria and mammalian) and interactions between cells (microbial and host-microbe).   |
| Gerhard Mayer | HITS gGmbH | Remotely (virtually) <br/> BioPAX, CellML, COMBINE Archive, Multicellular modeling, NeuroML, OMEX Metadata, SBGN, SBML, SBOL and SBOL Visual, SED-ML, identifiers.org <br/>EDITH (European Virtual Human Twin project) |
| Pedro Mendes | UConn Health | Remotely (virtually) <br/> OMEX Metadata, SBML, SED-ML <br/>COPASI |
| Blinov Michael | UConn Health | Remotely (virtually) <br/> BioPAX, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBGN, SBML, SED-ML <br/>VCell, BioNetGen, SBML |
| Tom Mitchell | Raytheon BBN Technologies | Remotely (virtually) <br/> SBOL and SBOL Visual <br/>pySBOL3, SBOL-utilities |
| Ion Moraru | UConn Health | In person <br/> COMBINE Archive, Multicellular modeling, SBML, SED-ML <br/>http://vcell.org/, http://biosimulations.org/ |
| Sébastien Moretti | SIB Swiss Institute of Bioinformatics | Remotely (virtually) <br/> SBML <br/>https://www.metanetx.org/ |
| Robert Mueller | TU Dresden | Remotely (virtually) <br/> Multicellular modeling, Morpheus  |
| Chris Myers | University of Colorado Boulder | In person <br/> COMBINE Archive, SBML, SBOL and SBOL Visual, SED-ML <br/>SynBioHub, SynBioSuite, iBioSim |
| David Nickerson | Auckland Bioengineering Institute, University of Auckland | In person <br/> CellML, COMBINE Archive, OMEX Metadata, SED-ML <br/>Physiome Model Repository; Center for Reproducible Biomedical Modeling; Physiome journal |
| Alexander  Patrie | Uconn Health | Remotely (virtually) <br/> COMBINE Archive, OMEX Metadata, SBML, SED-ML <br/>Biosimulators: https://www.biosimulators.org |
| Tim Rudge | newcastle university | Remotely (virtually) <br/> Multicellular modeling, SBML, SBOL and SBOL Visual, SED-ML <br/>Flapjack, LOICA |
| Sai Samineni  | University of Colorado Boulder  | Remotely (virtually) <br/> SBOL and SBOL Visual, LabOp <br/>LabOp, Army Data Standards |
| Jim Schaff | UConn Health | Remotely (virtually) <br/> COMBINE Archive, Multicellular modeling, OMEX Metadata, SBML, SED-ML <br/>SBML Spatial |
| T.j. Sego | University of Florida | In person <br/> CellML, COMBINE Archive, Multicellular modeling, SBGN, SBML, SED-ML <br/>CompuCell3D, Tellurium, Tissue Forge |
| Bilal Shaikh | DeepOrigin | Remotely (virtually) <br/> CellML, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBML, SED-ML <br/>Biosimulations  |
| Rahuman Sheriff | EMBL-EBI | Remotely (virtually) <br/> CellML, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBGN, SBML, SED-ML, FROG <br/>SBML, BioModels, FROG, FAIRSPACE |
| Woosub Shin | University of Auckland | Remotely (virtually) <br/> COMBINE Archive, OMEX Metadata, SBML  |
| Lucian Smith | University of Washington | In person <br/> CellML, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBML, SED-ML <br/>Antimony, Tellurium |
| Hugh Sorby | Auckland Bioengineering Institute | In person <br/> CellML, COMBINE Archive, SED-ML  |
| Ryan Spangler | Altos Labs | In person <br/> Multicellular modeling <br/>Vivarium |
| Jörn Starruß | Technische Universität Dresden | Remotely (virtually) <br/> COMBINE Archive, Multicellular modeling, SBML, SED-ML  |
| Nguyen Tung | EMBL-EBI | Remotely (virtually) <br/> BioPAX, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBGN, SBML, SED-ML <br/>BioModels, COMBINE Archive 2.0 |
| Lorenzo Veschini | King's Collge London | Remotely (virtually) <br/> Multicellular modeling, OMEX Metadata, SBML  |
| Gonzalo andrés Vidal peña | Newcastle University | Remotely (virtually) <br/> SBOL and SBOL Visual <br/>https://github.com/RudgeLab/PUDU https://github.com/RudgeLab/LOICA |
| Carolus Vitalis | University of Colorado Boulder | Remotely (virtually) <br/> SBOL and SBOL Visual  |
| Khoi Vo | University of California, Riverside | In person <br/> BioPAX, CellML, COMBINE Archive, Multicellular modeling, NeuroML, OMEX Metadata, SBGN, SBML, SBOL and SBOL Visual, SED-ML <br/>Multiscale modeling of blood clot |
| Daniel Weindl | Helmholtz Munich | Remotely (virtually) <br/> SBML, SED-ML, PEtab <br/>https://github.com/AMICI-dev/AMICI/ https://github.com/ICB-DCM/pyPESTO/ https://petab.readthedocs.io/en/latest/ |
| Felipe Xavier buson | Cambridge University | Remotely (virtually) <br/> SBOL and SBOL Visual  |
| Guillermo Yanez feliu | Newcastle University | Remotely (virtually) <br/> Multicellular modeling, Flapjack <br/>Multicellular pattern formation, computational synthetic biology, microscopy, microfluidics and image analysis. |
| Fengkai Zhang | NIAID/NIH | In person <br/> COMBINE Archive, Multicellular modeling, SBGN, SBML, SED-ML <br/>Simmune |


