<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 4.8.0 for Hugo"><meta name=author content="COMBINE"><meta name=description content><link rel=alternate hreflang=en-us href=/author/combine-2023/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#2962ff"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark disabled><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.css integrity="sha512-1xoFisiGdy9nvho8EgXuXvnpR5GAMSjFwp40gSRE3NwdUdIMIKuPa7bqoUhLD0O/5tPNhteAsE5XyyMi5reQVA==" crossorigin=anonymous><script src=https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin=anonymous async></script><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=/css/wowchemy.css><link rel=alternate href=/author/combine-2023/index.xml type=application/rss+xml title="Combine: Coordinating standards for modeling in Biology"><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/images/icon_hu506751a9b1968827413e70b30c336728_17544_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/images/icon_hu506751a9b1968827413e70b30c336728_17544_192x192_fill_lanczos_center_2.png><link rel=canonical href=/author/combine-2023/><meta property="twitter:card" content="summary"><meta property="og:site_name" content="Combine: Coordinating standards for modeling in Biology"><meta property="og:url" content="/author/combine-2023/"><meta property="og:title" content="COMBINE 2023 | Combine: Coordinating standards for modeling in Biology"><meta property="og:description" content><meta property="og:image" content="/images/logo.svg"><meta property="twitter:image" content="/images/logo.svg"><meta property="og:locale" content="en-us"><title>COMBINE 2023 | Combine: Coordinating standards for modeling in Biology</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents><script>window.wcDarkLightEnabled=true;</script><script>const isSiteThemeDark=false;</script><script src=/js/load-theme.js></script><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/><img src=/images/logo.svg alt="Combine: Coordinating standards for modeling in Biology"></a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/><img src=/images/logo.svg alt="Combine: Coordinating standards for modeling in Biology"></a></div><div class="navbar-collapse main-menu-item collapse justify-content-end" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/people><span>People</span></a></li><li class=nav-item><a class=nav-link href=/standards><span>Standards</span></a></li><li class=nav-item><a class=nav-link href=/events><span>Events</span></a></li><li class=nav-item><a class=nav-link href=/documents><span>Documents</span></a></li><li class=nav-item><a class=nav-link href=/examples><span>Examples</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav><section id=profile-page class=pt-5><div class=container><div class="row justify-content-center people-widget"></div><div class="row justify-content-center people-widget"></div><div class=row><div class="col-12 col-lg-4"><h1 style=color:#1f646b>COMBINE 2023</h1></div></div><div class=row><div class="col-12 col-lg-4"><h2 style=color:#8dc742>Conference Date</h2><b>5-8, October 2023</b><h2 style=color:#8dc742>Conference Location</h2><b>Farmington, CT</b><h2 style=color:#8dc742>Important Dates</h2><div>Breakouts, talks and tutorials submission deadline</div><b>September 1, 2023</b><p></p><div>Lightning talk and poster submission deadline</div><b>September 15, 2023</b><p></p><div>Notification of acceptance of breakouts and tutorials</div><b>September 7, 2023</b><p></p><div>DEADLINE for student travel support applications</div><b>September 1, 2023</b><p></p><h2 style=color:#8dc742>Important Links</h2><div class=description><a href=https://forms.gle/7pbiYps84yUvEbsY9 style=color:#1f646b>Registration</a></div><div class=description><a href=https://forms.gle/i1ENK9NicXBshhUM7 style=color:#1f646b>Break-outs and tutorials submission</a></div><div class=description><a href=https://forms.gle/3ej8xnxQotzz6ZV99 style=color:#1f646b>Talks, lightning talks, and poster submission</a></div><div class=description><a href=https://forms.gle/whr1MusnSEFZD3vy8 style=color:#1f646b>Application for student travel support</a></div></div><div class="col-12 col-lg-8"><p></p><img src=/images/combine2023/combine23-hartford.png alt=demo class=img-responsive>
<font color=red>NEWS: list of attendees as of September 9 is posted below</font><br><p>The &ldquo;<strong>Computational Modeling in Biology</strong>&rdquo; Network (<strong>COMBINE</strong>) is an initiative to coordinate the development of the various community standards and formats in systems biology and related fields. <strong>COMBINE 2023</strong> will be a workshop-style event hosted at the <a href=https://health.uconn.edu/cell-analysis-modeling/ target=_blank rel=noopener>Center for Cell Analysis and Modeling</a> at the <a href=https://medicine.uconn.edu/ target=_blank rel=noopener>University of Connecticut School of Medicine</a>, Farmington, CT, USA. The meeting will be held in October 2023, closely aligned with the dates of the <a href=https://icsb2023.bioscience-ct.net/ target=_blank rel=noopener><strong>International Conference on Systems Biology (ICSB)</strong></a> October 8-12 in Hartford, CT. Tutorials at the last day (Sunday October 8th) will take place at the ICSB 2023 venue - in the convention center. The meeting days will include talks about the COMBINE standards and associated or related standardization efforts, presentations of tools using these standards, breakout sessions for detailed discussions as well as tutorials. There are no dedicated poster sessions, but paricipants are encouraged to bring posters - poster boards will be provided next to meeting places. Some time each day will be left for community discussion and wrap-ups of breakouts and advertisements for following breakouts. It will be primarily an in-person meeting, with individual breakout sessions responsible for enabling remote participation as needed.</p><p>Local organizers are Michael Blinov (<a href=mailto:blinov@uchc.edu>blinov@uchc.edu</a>) and Ion Moraru (<a href=mailto:moraru@uchc.edu>moraru@uchc.edu</a>).</p><p><font color=red>Students may apply for a travel support from NSF!!! Link is on the LEFT!</font></p><h3>A tentative schedule</h3><p><img src=/images/combine2023/schedule.png width=100% alt=schedule.png>Note that many events are scheduled somewhat spontaneously at these events; keep an eye out here or on the COMBINE slack for last-minute changes and additions.</p><h3>Workshop Location</h3><p>COMBINE 2023 takes place at the <a href=https://health.uconn.edu/cell-analysis-modeling/ target=_blank rel=noopener>Center for Cell Analysis and Modeling (CCAM)</a> in Farmington, CT. <strong>COMBINE 2023</strong> will take place in the Cell and Genome Sciences building, at <a href=https://goo.gl/maps/Ms25yakj7NU7dFsv5 target=_blank rel=noopener>400 Farmington Ave, Farmington, CT 06119</a>. If the doors are locked, please knock, email <a href=mailto:blinov@uchc.edu>blinov@uchc.edu</a> or <a href=mailto:moraru@uchc.edu>moraru@uchc.edu</a>, or ping us on the <a href=https://combine-workspace.slack.com target=_blank rel=noopener>COMBINE slack channel</a>.</p><img src=/images/combine2023/ccam3.png width=100% alt=ccam3.png><h3>Arrival and Transportation</h3><p>The closest airport is the <a href=https://bradleyairport.com/ target=_blank rel=noopener>Bradley International Airport (BDL)</a>. The recommended transportation is to take Uber/Lyft to 400 Farmington Ave, Farmington, CT. It takes about 25 minutes and costs about $40. You may rent a car at BDL - all parking at the workshop location and hotel are free. A less expensive but long travel from BDL is to take Bradley Flyer Bus (#30) from BDL to Central Row North Side at Old State House Station, and then take a <a href=https://goo.gl/maps/9BpUPEqBUb5Zgrnm7 target=_blank rel=noopener>bus 66T</a> from Main St & Asylum St to 400 Farmington Ave. It takes 1 to 3 hours depending on schedules.</p><h3>Accomodations and meals</h3><p>The closest hotel to the CCAM is the <a href=https://www.hilton.com/en/hotels/hfdfmhw-homewood-suites-hartford-farmington/ target=_blank rel=noopener>Homewood Suites by Hilton Hartford-Farmington</a> at <a href=https://goo.gl/maps/jMK2dqEYqKSDf7vC7 target=_blank rel=noopener>2 Farm Glen Boulevard, Farmington, Connecticut, 06032</a>; it&rsquo;s <a href=https://goo.gl/maps/D5j5SQnSLiRQFaKx9 target=_blank rel=noopener>7 minutes walk</a> between there and the venue. A University rate of $154/night will be provided upon request. There are more hotels in the area, but any other hotel will require a car.</p><p>The hotel serves hot breakfast. There is a <a href=https://butchersandbakers.com/ target=_blank rel=noopener>Butchers and Bakers restaurant</a> 15 minutes walk, but no food stores within walking distance. We will serve food at the venue, depending on the sponsors it may be free or for a nominal fee.</p><h3>Attendees</h3><p>52 registered participants as of September 12</p><table><thead><tr><th>Name<hr>Affiliation</th><th>Attendance</th><th>Interested<hr>Projects</th></tr></thead><tbody><tr><td><b>Amin Boroomand</b><br>Woods Hole institute</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;Multicellular modeling<hr>-</td></tr><tr><td><b>Amir Mahari</b><br>University of Arkansas</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;SBGN;Multicellular modeling<hr>I focus on the modeling and simulation of intercellular signaling pathways within melanoma cancer cells. This study holds significance due to the intricate nature of signaling cascades that regulate various cellular processes in melanoma progression. To facilitate this investigation, I employ PySB (Python Systems Biology), a powerful computational framework designed for creating, simulating, and analyzing mathematical models of biochemical systems. PySB&rsquo;s modular and user-friendly nature enables me to construct complex models of intercellular signaling, incorporating factors such as ligand-receptor interactions, protein phosphorylation, and gene expression. Through PySB&rsquo;s simulation capabilities, I can dynamically analyze the behavior of these pathways under different conditions, gaining insights into the underlying mechanisms that drive melanoma growth and metastasis. Ultimately, this research using PySB contributes to an enhanced understanding of melanoma biology, potentially paving the way for novel therapeutic interventions and personalized treatment strategies in combating this aggressive form of cancer.</td></tr><tr><td><b>Augustin Luna</b><br>Harvard Medical School</td><td>remotely</td><td>SBGN;BioPAX<hr><a href=https://www.pathwaycommons.org/;>https://www.pathwaycommons.org/;</a> <a href=https://biofactoid.org/>https://biofactoid.org/</a></td></tr><tr><td><b>Bartholomew Jardine</b><br>University of Washington</td><td>remotely</td><td>SBML;SED-ML, OMEX, COMBINE;Ontologies (SBO, KiSAO);Multicellular modeling<hr>Software tools for creating, editing, and simulating SBML complient modelss</td></tr><tr><td><b>Bharat Mishra</b><br>PhD</td><td></td><td>SBML;SED-ML, OMEX, COMBINE;CellML;Ontologies (SBO, KiSAO);Multicellular modeling<hr><a href=https://www.uab.edu/cores/ircp/bds>https://www.uab.edu/cores/ircp/bds</a></td></tr><tr><td><b>Chris Myers</b><br>University of Colorado Boulder</td><td>Oct 5 Oct 6 Oct 7</td><td>SBML;SED-ML, OMEX, COMBINE;SBOL, SBOL visual<hr>SynBioHub/SynBioSuite/iBioSim</td></tr><tr><td><b>Dagmar Waltemath</b><br>University Medicine Greifswald</td><td>remotely</td><td>SED-ML, OMEX, COMBINE;Ontologies (SBO, KiSAO)<hr>-</td></tr><tr><td><b>Dan Vasilescu</b><br>UCHC</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;SED-ML, OMEX, COMBINE;BioPAX;Ontologies (SBO, KiSAO)<hr>-</td></tr><tr><td><b>Daniel Ajuzie</b><br>undergraduate</td><td>Oct 5 Oct 6 Oct 7</td><td>SBML;SED-ML, OMEX, COMBINE;CellML;NeuroML<hr>Multi-phenotype Optimization for Dynamic Modeling</td></tr><tr><td><b>Daniel Ajuzie</b><br>undergraduate</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;SED-ML, OMEX, COMBINE;SBOL, SBOL visual;CellML;Ontologies (SBO, KiSAO);Multicellular modeling<hr>Ensemble Modeling; Model Optimization and Calibration</td></tr><tr><td><b>David Nickerson</b><br>Auckland Bioengineering Institute, University of Auckland</td><td>remotely</td><td>SED-ML, OMEX, COMBINE;CellML;Ontologies (SBO, KiSAO);OMEX Metadata; FAIR indicators for models; repositories<hr>-</td></tr><tr><td><b>Diego Jahn</b><br>TUD Dresden University of Technology, Center for Information Services and High Performance Computing (ZIH)</td><td>remotely</td><td>Multicellular modeling<hr>MorpheusML, MorpheusML Model Repository (<a href=https://morpheus.gitlab.io>https://morpheus.gitlab.io</a>), Multicellular Modeling</td></tr><tr><td><b>Difei Tang</b><br>University of Pittsburgh</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;BioPAX;SBOL, SBOL visual;Ontologies (SBO, KiSAO)<hr>GUI for DySE framework</td></tr><tr><td><b>Egils Stalidzans</b><br>PhD</td><td></td><td>SBML;SBOL, SBOL visual<hr>ODE-based kinetic modeling, constraint based stoichiometric modeling</td></tr><tr><td><b>Elebeoba May</b><br>University of Wisconsin-Madison</td><td></td><td>SBML;Multicellular modeling;Visualization<hr>biochemical/bionetwork ODE models (metabolism, signal transduction, gene networks), multiscale models of host-pathogen systems, multiscale models of microbial communities, modeling synthetic bio systems</td></tr><tr><td><b>Eran Agmon</b><br>UConn Health</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SED-ML, OMEX, COMBINE;Multicellular modeling<hr>Vivarium: <a href=https://vivarium-collective.github.io>https://vivarium-collective.github.io</a></td></tr><tr><td><b>Fengkai Zhang</b><br>NIH</td><td></td><td>SBML;SED-ML, OMEX, COMBINE;SBGN<hr>-</td></tr><tr><td><b>Frank T. Bergmann</b><br>BioQUANT, Heidelberg University</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;SED-ML, OMEX, COMBINE;SBGN<hr>COPASI (<a href=https://copasi.org>https://copasi.org</a>), basico (<a href=https://basico.readthedocs.io/),>https://basico.readthedocs.io/),</a> libSBML / libSEDML / libCombine</td></tr><tr><td><b>Gaoxiang Zhou</b><br>University of Pittsburgh</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;BioPAX;SBOL, SBOL visual;CellML;Multicellular modeling<hr><a href=https://www.nmzlab.pitt.edu/people/gaoxiang-zhou,>https://www.nmzlab.pitt.edu/people/gaoxiang-zhou,</a> <a href=https://github.com/pitt-miskov-zivanov-lab,>https://github.com/pitt-miskov-zivanov-lab,</a> <a href=https://melody-biorecipe.readthedocs.io>https://melody-biorecipe.readthedocs.io</a></td></tr><tr><td><b>Gerhard Mayer</b><br>HITS (Heidelberg Institute for Theoretical Studies) gGmbH, Heidelberg</td><td>remotely</td><td>SED-ML, OMEX, COMBINE;Multicellular modeling<hr>EDITH (Ecosystem Digital Twins in Healthcare); <a href=https://www.edith-csa.eu>https://www.edith-csa.eu</a></td></tr><tr><td><b>Herbert M Sauro</b><br>University of Washington</td><td>remotely</td><td>SBML;SED-ML, OMEX, COMBINE;Multicellular modeling<hr>SBML, roadrunner etc</td></tr><tr><td><b>James A. Glazier</b><br>Indiana University</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>Multicellular modeling<hr>CompuCell3D (<a href=https://compucell3d.org/),>https://compucell3d.org/),</a> Virtual Cornea, IMAG/MSM Working Group on Multiscale Modeling and Viral Pandemics (<a href=https://www.imagwiki.nibib.nih.gov/working-groups/multiscale-modeling-and-viral-pandemics>https://www.imagwiki.nibib.nih.gov/working-groups/multiscale-modeling-and-viral-pandemics</a>)</td></tr><tr><td><b>Jim Schaff</b><br>Contractor - UConn Health</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SED-ML, OMEX, COMBINE;Multicellular modeling<hr>Virtual Cell Project (vcell.org), Reproducible Biological Modeling (reproduciblebiomodels.org)</td></tr><tr><td><b>John Gennari</b><br>University of Washington</td><td>Oct 5 Oct 6 Oct 7</td><td>SBML;SED-ML, OMEX, COMBINE;BioPAX;CellML;Ontologies (SBO, KiSAO)<hr>Center for Reproducible Biomedical Modeling</td></tr><tr><td><b>Juliano Ferrari Gianlupi</b><br>Postdoctoral Scholar, UTHSC</td><td></td><td>SBML;CellML;Multicellular modeling<hr>PhenoCellPy <a href=https://www.biorxiv.org/content/10.1101/2023.04.12.535625v2.abstract;>https://www.biorxiv.org/content/10.1101/2023.04.12.535625v2.abstract;</a> Translating PhysiCell specification into CompuCell3D simulation <a href=https://github.com/JulianoGianlupi/pcxml2cc3d>https://github.com/JulianoGianlupi/pcxml2cc3d</a></td></tr><tr><td><b>Jörn Starruß</b><br>Technische Universität Dresden, Germany</td><td>remotely</td><td>SBML;SED-ML, OMEX, COMBINE;CellML;Multicellular modeling<hr>MorpheusML, <a href=https://morpheus.gitlab.io>https://morpheus.gitlab.io</a>, Multicellular modeling, <a href=https://MultiCellML.org>https://MultiCellML.org</a>, SBML-Spatial, PEtab-MS, <a href=https://gitlab.com/fitmulticell/fit>https://gitlab.com/fitmulticell/fit</a></td></tr><tr><td><b>Jürgen Pahle</b><br>Heidelberg University</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>-<hr>Copasi, CoRC</td></tr><tr><td><b>Lea Seep</b><br>University Bonn, IRU-MLS</td><td></td><td>SBML;Ontologies (SBO, KiSAO)<hr>-</td></tr><tr><td><b>Lucian Smith</b><br>University of Washington</td><td>remotely</td><td>SBML;SED-ML, OMEX, COMBINE;Ontologies (SBO, KiSAO);Multicellular modeling<hr>Antimony, Roadrunner, Tellurium, SBML, SED-ML</td></tr><tr><td><b>Luis Fonseca</b><br>University of Florida</td><td>remotely</td><td>SBML;SBGN;CellML;NeuroML;Multicellular modeling;ABM<hr>-</td></tr><tr><td><b>Lukas Buecherl</b><br>University of Colorado Boulder</td><td></td><td>SBML;SED-ML, OMEX, COMBINE;SBOL, SBOL visual<hr>-</td></tr><tr><td><b>Lutz Brusch</b><br>Technische Universität Dresden, Germany</td><td>remotely</td><td>Multicellular modeling<hr>Multicellular modeling, <a href=https://MultiCellML.org>https://MultiCellML.org</a>, MorpheusML, <a href=https://morpheus.gitlab.io>https://morpheus.gitlab.io</a>, SBML-Spatial, PEtab-MS, <a href=https://gitlab.com/fitmulticell/fit,>https://gitlab.com/fitmulticell/fit,</a> FAIRSPACE</td></tr><tr><td><b>Maren Philipps</b><br>University of Bonn</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>-<hr>PEtab, pyPESTO</td></tr><tr><td><b>Mauro Silberberg</b><br>Universidad de Buenos Aires, Facultad de Ciencias Exactas y Naturales, Departamento de Física, y CONICET - Universidad de Buenos Aires, Instituto de Física de Buenos Aires (IFIBA). Buenos Aires, Argentina</td><td>remotely</td><td>SBML;SED-ML, OMEX, COMBINE;CellML;NeuroML<hr>poincaré (github.com/maurosilber/poincare) and SimBio (github.com/hgrecco/simbio), which are Python libraries for definition and simulation of dynamical systems.</td></tr><tr><td><b>Michael Blinov</b><br>Center for Cell Analysis and Modeling, UConn Health</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;SED-ML, OMEX, COMBINE;SBGN;BioPAX;Ontologies (SBO, KiSAO);Multicellular modeling<hr>VCell, BNGLViz, MolClustPy</td></tr><tr><td><b>Mustafa Ozen</b><br>Altos Labs</td><td></td><td>SED-ML, OMEX, COMBINE;CellML;NeuroML;Multicellular modeling<hr>-</td></tr><tr><td><b>Natasa Miskov-Zivanov</b><br>University of Pittsburgh</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;SED-ML, OMEX, COMBINE;SBGN;BioPAX;SBOL, SBOL visual;CellML;NeuroML;Ontologies (SBO, KiSAO);Multicellular modeling<hr>-</td></tr><tr><td><b>Nilesh Kumar</b><br>PhD</td><td></td><td>SBML;SED-ML, OMEX, COMBINE;CellML<hr>-</td></tr><tr><td><b>Olga Krebs</b><br>Heidelberg Institute for Theoretical Studies HITS</td><td></td><td>SED-ML, OMEX, COMBINE;SBOL, SBOL visual;Ontologies (SBO, KiSAO);BioProtocols<hr>FAIRDOM, LiSyM Cancer, MESI-STRAT, PoLiMeR, deNBI</td></tr><tr><td><b>Paola Vera-Licona</b><br>UConn Health</td><td>Oct 5 Oct 6 Oct 7</td><td>SBML;SED-ML, OMEX, COMBINE;CellML;Multicellular modeling<hr>NETISCE (NETwork-drIven analysiS of CEllular reprogramming) <a href=http://veraliconalab.org/Netisce/index.html>http://veraliconalab.org/Netisce/index.html</a></td></tr><tr><td><b>Pedro Mendes</b><br>University of Connecticut School of Medicine</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;SED-ML, OMEX, COMBINE;Ontologies (SBO, KiSAO);Multicellular modeling<hr>-</td></tr><tr><td><b>Prashant Vaidyanathan</b><br>Oxford Biomedica</td><td>remotely</td><td>SBOL, SBOL visual<hr>-</td></tr><tr><td><b>Rahuman Sheriff</b><br>European Bioinformatics Institute (EMBL-EBI)</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;SED-ML, OMEX, COMBINE;SBGN;Ontologies (SBO, KiSAO);Multicellular modeling;FROG<hr>BioModels <a href=https://www.ebi.ac.uk/biomodels>https://www.ebi.ac.uk/biomodels</a></td></tr><tr><td><b>Sebastien Moretti</b><br>SIB Swiss Institute of Bioinformatics</td><td>remotely</td><td>SBML;Ontologies (SBO, KiSAO)<hr>-</td></tr><tr><td><b>Sikao Guo</b><br>PostDoc</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;BioPAX;CellML;NeuroML;Multicellular modeling<hr>I am currently working on developing and parallelizing NERDSS (Structure-Resolved Reaction-Diffusion Simulation Software). You can find the GitHub page at <a href=https://github.com/mjohn218/NERDSS>https://github.com/mjohn218/NERDSS</a>.</td></tr><tr><td><b>T.J. Sego</b><br>University of Florida</td><td>Oct 5 Oct 6 Oct 7 Oct 8</td><td>SBML;Multicellular modeling<hr>CompuCell3D, Tissue Forge, <a href=https://directory.ufhealth.org/sego-t-j>https://directory.ufhealth.org/sego-t-j</a></td></tr></tbody></table><h2><a href=https://forms.gle/7pbiYps84yUvEbsY9 target=_blank rel=noopener><span style=color:#8dc742>Registration</span> <i class="fas fa-chevron-right" aria-hidden=true></i></a></h2>Registration for the meeting is free. Please register at the link above as soon as possible. This will help us plan the schedule and match your interests to the timing of the breakouts, etc. Note, only registered attendees will be sent information related to the meeting.<h2><a href=https://forms.gle/i1ENK9NicXBshhUM7 target=_blank rel=noopener><span style=color:#8dc742>Call for Breakout Sessions and Tutorials</span> <i class="fas fa-chevron-right" aria-hidden=true></i></a></h2>All attendees can suggest breakout sessions for hacking and/or detailed discussions of certain aspects of one or several of the COMBINE standard(s), metadata and semantic annotations (format-specific or overarching), application and implementations of the COMBINE standards, or any other topic relevant for the COMBINE community. The topics for those breakout sessions, and the time slots which would suit their communities can be submitted via the link above. Note, breakout session organisers will be responsible for creating and hosting their own online sessions, if required.<h2><a href=https://forms.gle/3ej8xnxQotzz6ZV99 target=_blank rel=noopener><span style=color:#8dc742>Call for Talks, Lightning Talks and Posters</span> <i class="fas fa-chevron-right" aria-hidden=true></i></a></h2>Requests for a talk (40 minutes), lightning talk (5 min max.) and/or poster can be submitted via the link above. Please use several forms if you want to submit abstracts on different topics. The submission deadline is outlined above. Talks will take place during the community session and posters will be displayed throughout the meeting.<h2 style=color:#8dc742>Topics of Interest</h2><ul><li>Data exchange, pipelines and model standards for systems and synthetic biology</li><li>Visualization and graphical notation standards for systems and synthetic biology</li><li>Standards for sharing and analysing biological pathway data</li><li>Standards for computational biological models and modelling support</li><li>Metadata description and model annotation in COMBINE standard formats</li><li>Implementation of COMBINE standards in tools, databases and other resources</li><li>Integrated model and data management for systems and synthetic biology</li><li>Standardization of Artificial Intelligence approaches in biological modelling</li><li>Emerging standardization needs and multicellular modeling</li><li>Community aspects of COMBINE</li></ul></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script src=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.js integrity="sha512-SeiQaaDh73yrb56sTW/RgVdi/mMqNeM2oBwubFHagc5BkixSpP1fvqF47mKzPGWYSSy4RwbBunrJBQ4Co8fRWA==" crossorigin=anonymous></script><script>const code_highlighting=true;</script><script>const search_config={"indexURI":"/index.json","minLength":1,"threshold":0.3};const i18n={"no_results":"No results found","placeholder":"Search...","results":"results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'talk':"Talks",'slides':"Slides"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/js/wowchemy.min.4c2bca31150ce93c5a5e43b8a50f22fd.js></script><div class=container><footer class=site-footer><p class=powered-by></p><p class=powered-by>Published with
<a href=https://wowchemy.com target=_blank rel=noopener>Wowchemy</a> —
the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.
<span class=float-right aria-hidden=true><a href=# class=back-to-top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div></body></html>