---
title: COMBINE 2023

# Is this the primary user of the site?
superuser: false

conf_date: 5-8, October 2023

conf_location: Farmington, CT

important_dates:
- name: Breakouts, talks and tutorials submission deadline
  date: September 1, 2023
- name: Lightning talk and poster submission deadline
  date: September 15, 2023
- name: Notification of acceptance of breakouts and tutorials
  date: September 7, 2023
- name: DEADLINE for student travel support applications
  date: September 1, 2023

important_links:
#- name: Agenda
#  link: 
- name: Registration
  link: https://forms.gle/7pbiYps84yUvEbsY9
- name: Break-outs and tutorials submission
  link: https://forms.gle/i1ENK9NicXBshhUM7
- name: Talks, lightning talks, and poster submission
  link: https://forms.gle/3ej8xnxQotzz6ZV99
- name: Application for student travel support
  link: https://forms.gle/whr1MusnSEFZD3vy8

information:
- name: Registration
  description: Registration for the meeting is free. Please register at the link above as soon as possible. This will help us plan the schedule and match your interests to the timing of the breakouts, etc. Note, only registered attendees will be sent information related to the meeting.
  link: https://forms.gle/7pbiYps84yUvEbsY9
- name: Call for Breakout Sessions and Tutorials
  description: All attendees can suggest breakout sessions for hacking and/or detailed discussions of certain aspects of one or several of the COMBINE standard(s), metadata and semantic annotations (format-specific or overarching), application and implementations of the COMBINE standards, or any other topic relevant for the COMBINE community. The topics for those breakout sessions, and the time slots which would suit their communities can be submitted via the link above. Note, breakout session organisers will be responsible for creating and hosting their own online sessions, if required.
  link: https://forms.gle/i1ENK9NicXBshhUM7
- name: Call for Talks, Lightning Talks and Posters
  description: Requests for a talk (40 minutes), lightning talk (5 min max.) and/or poster can be submitted via the link above. Please use several forms if you want to submit abstracts on different topics. The submission deadline is outlined above. Talks will take place during the community session and posters will be displayed throughout the meeting.
  link: https://forms.gle/3ej8xnxQotzz6ZV99


topics:
- Data exchange, pipelines and model standards for systems and synthetic biology
- Visualization and graphical notation standards for systems and synthetic biology
- Standards for sharing and analysing biological pathway data
- Standards for computational biological models and modelling support
- Metadata description and model annotation in COMBINE standard formats
- Implementation of COMBINE standards in tools, databases and other resources
- Integrated model and data management for systems and synthetic biology
- Standardization of Artificial Intelligence approaches in biological modelling
- Emerging standardization needs and multicellular modeling
- Community aspects of COMBINE

# Organizational groups that you belong to (for People widget)
#   Set this to `[]` or comment out if you are not using People widget.
user_groups:
- Events

---
<img src="/images/combine2023/combine23-hartford.png" alt="demo" class="img-responsive">
<font color = red>NEWS September 12: 
  <ul>
  <li>52 [attendies](#attendies) </li>
    <li>Preliminary list of 8 [breakouts](#breakouts) and 6 [tutorials](#tutorials)</li>
    </ul>
  </font>

The "**Computational Modeling in Biology**" Network (**COMBINE**) is an initiative to coordinate the development of the various community standards and formats in systems biology and related fields. **COMBINE 2023** will be a workshop-style event hosted at the [Center for Cell Analysis and Modeling](https://health.uconn.edu/cell-analysis-modeling/) at the [University of Connecticut School of Medicine](https://medicine.uconn.edu/), Farmington, CT, USA. The meeting will be held in October 2023, closely aligned with the dates of the [**International Conference on Systems Biology (ICSB)**](https://icsb2023.bioscience-ct.net/) October 8-12 in Hartford, CT. Tutorials at the last day (Sunday October 8th) will take place at the ICSB 2023 venue - in the convention center. The meeting days will include talks about the COMBINE standards and associated or related standardization efforts, presentations of tools using these standards, breakout sessions for detailed discussions as well as tutorials. There are no dedicated poster sessions, but paricipants are encouraged to bring posters - poster boards will be provided next to meeting places. Some time each day will be left for community discussion and wrap-ups of breakouts and advertisements for following breakouts. It will be primarily an in-person meeting, with individual breakout sessions responsible for enabling remote participation as needed.

Local organizers are Michael Blinov (blinov@uchc.edu) and Ion Moraru (moraru@uchc.edu). 

<font color = red>Students may apply for a travel support from NSF!!! Link is on the LEFT! </font>

<h3>A tentative schedule</h3>

<img src="/images/combine2023/schedule.png" width="100%" alt="schedule.png">Note that many events are scheduled somewhat spontaneously at these events; keep an eye out here or on the COMBINE slack for last-minute changes and additions.

<h3>Workshop Location</h3>

COMBINE 2023 takes place at the [Center for Cell Analysis and Modeling (CCAM)](https://health.uconn.edu/cell-analysis-modeling/) in Farmington, CT. **COMBINE 2023** will take place in the Cell and Genome Sciences building, at [400 Farmington Ave, Farmington, CT 06119](https://goo.gl/maps/Ms25yakj7NU7dFsv5). If the doors are locked, please knock, email blinov@uchc.edu or moraru@uchc.edu, or ping us on the [COMBINE slack channel](https://combine-workspace.slack.com).

<img src="/images/combine2023/ccam3.png" width="100%" alt="ccam3.png">

<h3>Arrival and Transportation</h3>

The closest airport is the [Bradley International Airport (BDL)](https://bradleyairport.com/). The recommended transportation is to take Uber/Lyft to 400 Farmington Ave, Farmington, CT. It takes about 25 minutes and costs about $40. You may rent a car at BDL - all parking at the workshop location and hotel are free. A less expensive but long travel from BDL is to take Bradley Flyer Bus (#30) from BDL to Central Row North Side at Old State House Station, and then take a [bus 66T](https://goo.gl/maps/9BpUPEqBUb5Zgrnm7) from Main St & Asylum St to 400 Farmington Ave. It takes 1 to 3 hours depending on schedules. 

<h3>Accomodations and meals</h3>

The closest hotel to the CCAM is the [Homewood Suites by Hilton Hartford-Farmington](https://www.hilton.com/en/hotels/hfdfmhw-homewood-suites-hartford-farmington/) at [2 Farm Glen Boulevard, Farmington, Connecticut, 06032](https://goo.gl/maps/jMK2dqEYqKSDf7vC7); it's [7 minutes walk](https://goo.gl/maps/D5j5SQnSLiRQFaKx9) between there and the venue. A University rate of $154/night will be provided upon request. There are more hotels in the area, but any other hotel will require a car.

The hotel serves hot breakfast. There is a [Butchers and Bakers restaurant](https://butchersandbakers.com/) 15 minutes walk, but no food stores within walking distance. We will serve food at the venue, depending on the sponsors it may be free or for a nominal fee.

<h3><a name="attendies">Attendees</a></h3>

52 registered participants as of September 12
| Name <hr> Affiliation  | Attendance | Interested <hr>  Projects |
|---|---|---|
| <b>Amin Boroomand</b> <br/> Woods Hole institute | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;Multicellular modeling <hr> - | 
| <b>Amir Mahari</b> <br/> University of Arkansas | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SBGN;Multicellular modeling <hr> I focus on the modeling and simulation of intercellular signaling pathways within melanoma cancer cells. This study holds significance due to the intricate nature of signaling cascades that regulate various cellular processes in melanoma progression. To facilitate this investigation, I employ PySB (Python Systems Biology), a powerful computational framework designed for creating, simulating, and analyzing mathematical models of biochemical systems. PySB's modular and user-friendly nature enables me to construct complex models of intercellular signaling, incorporating factors such as ligand-receptor interactions, protein phosphorylation, and gene expression. Through PySB's simulation capabilities, I can dynamically analyze the behavior of these pathways under different conditions, gaining insights into the underlying mechanisms that drive melanoma growth and metastasis. Ultimately, this research using PySB contributes to an enhanced understanding of melanoma biology, potentially paving the way for novel therapeutic interventions and personalized treatment strategies in combating this aggressive form of cancer. | 
| <b>Augustin Luna</b> <br/> Harvard Medical School | remotely | SBGN;BioPAX <hr> https://www.pathwaycommons.org/; https://biofactoid.org/ | 
| <b>Bartholomew Jardine</b> <br/> University of Washington | remotely | SBML;SED-ML, OMEX, COMBINE;Ontologies (SBO, KiSAO);Multicellular modeling <hr> Software tools for creating, editing, and simulating SBML complient modelss | 
| <b>Bharat Mishra</b> <br/> PhD |  | SBML;SED-ML, OMEX, COMBINE;CellML;Ontologies (SBO, KiSAO);Multicellular modeling <hr> https://www.uab.edu/cores/ircp/bds | 
| <b>Chris Myers</b> <br/> University of Colorado Boulder | Oct 5 Oct 6 Oct 7 | SBML;SED-ML, OMEX, COMBINE;SBOL, SBOL visual <hr> SynBioHub/SynBioSuite/iBioSim | 
| <b>Dagmar Waltemath</b> <br/> University Medicine Greifswald | remotely | SED-ML, OMEX, COMBINE;Ontologies (SBO, KiSAO) <hr> - | 
| <b>Dan Vasilescu</b> <br/> UCHC | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;BioPAX;Ontologies (SBO, KiSAO) <hr> - | 
| <b>Daniel Ajuzie</b> <br/> undergraduate | Oct 5 Oct 6 Oct 7 | SBML;SED-ML, OMEX, COMBINE;CellML;NeuroML <hr> Multi-phenotype Optimization for Dynamic Modeling | 
| <b>Daniel Ajuzie</b> <br/> undergraduate | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;SBOL, SBOL visual;CellML;Ontologies (SBO, KiSAO);Multicellular modeling <hr> Ensemble Modeling; Model Optimization and Calibration | 
| <b>David Nickerson</b> <br/> Auckland Bioengineering Institute, University of Auckland | remotely | SED-ML, OMEX, COMBINE;CellML;Ontologies (SBO, KiSAO);OMEX Metadata; FAIR indicators for models; repositories <hr> - | 
| <b>Diego Jahn</b> <br/> TUD Dresden University of Technology, Center for Information Services and High Performance Computing (ZIH) | remotely | Multicellular modeling <hr> MorpheusML, MorpheusML Model Repository (https://morpheus.gitlab.io), Multicellular Modeling | 
| <b>Difei Tang</b> <br/> University of Pittsburgh | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;BioPAX;SBOL, SBOL visual;Ontologies (SBO, KiSAO) <hr> GUI for DySE framework | 
| <b>Egils Stalidzans</b> <br/> PhD |  | SBML;SBOL, SBOL visual <hr> ODE-based kinetic modeling, constraint based stoichiometric modeling | 
| <b>Elebeoba May</b> <br/> University of Wisconsin-Madison |  | SBML;Multicellular modeling;Visualization <hr> biochemical/bionetwork ODE models (metabolism, signal transduction, gene networks), multiscale models of host-pathogen systems, multiscale models of microbial communities, modeling synthetic bio systems | 
| <b>Eran Agmon</b> <br/> UConn Health | Oct 5 Oct 6 Oct 7 Oct 8 | SED-ML, OMEX, COMBINE;Multicellular modeling <hr> Vivarium: https://vivarium-collective.github.io | 
| <b>Fengkai Zhang</b> <br/> NIH |  | SBML;SED-ML, OMEX, COMBINE;SBGN <hr> - | 
| <b>Frank T. Bergmann</b> <br/> BioQUANT, Heidelberg University | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;SBGN <hr> COPASI (https://copasi.org), basico (https://basico.readthedocs.io/), libSBML / libSEDML / libCombine | 
| <b>Gaoxiang Zhou</b> <br/> University of Pittsburgh | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;BioPAX;SBOL, SBOL visual;CellML;Multicellular modeling <hr> https://www.nmzlab.pitt.edu/people/gaoxiang-zhou, https://github.com/pitt-miskov-zivanov-lab, https://melody-biorecipe.readthedocs.io | 
| <b>Gerhard Mayer</b> <br/> HITS (Heidelberg Institute for Theoretical Studies) gGmbH, Heidelberg | remotely | SED-ML, OMEX, COMBINE;Multicellular modeling <hr> EDITH (Ecosystem Digital Twins in Healthcare); https://www.edith-csa.eu | 
| <b>Herbert M Sauro</b> <br/> University of Washington | remotely | SBML;SED-ML, OMEX, COMBINE;Multicellular modeling <hr> SBML, roadrunner etc | 
| <b>James A. Glazier</b> <br/> Indiana University | Oct 5 Oct 6 Oct 7 Oct 8 | Multicellular modeling <hr> CompuCell3D (https://compucell3d.org/), Virtual Cornea, IMAG/MSM Working Group on Multiscale Modeling and Viral Pandemics (https://www.imagwiki.nibib.nih.gov/working-groups/multiscale-modeling-and-viral-pandemics) | 
| <b>Jim Schaff</b> <br/> Contractor - UConn Health | Oct 5 Oct 6 Oct 7 Oct 8 | SED-ML, OMEX, COMBINE;Multicellular modeling <hr> Virtual Cell Project (vcell.org), Reproducible Biological Modeling (reproduciblebiomodels.org) | 
| <b>John Gennari</b> <br/> University of Washington | Oct 5 Oct 6 Oct 7 | SBML;SED-ML, OMEX, COMBINE;BioPAX;CellML;Ontologies (SBO, KiSAO) <hr> Center for Reproducible Biomedical Modeling | 
| <b>Juliano Ferrari Gianlupi</b> <br/> Postdoctoral Scholar, UTHSC |  | SBML;CellML;Multicellular modeling <hr> PhenoCellPy https://www.biorxiv.org/content/10.1101/2023.04.12.535625v2.abstract; Translating PhysiCell specification into CompuCell3D simulation https://github.com/JulianoGianlupi/pcxml2cc3d | 
| <b>Jörn Starruß</b> <br/> Technische Universität Dresden, Germany | remotely | SBML;SED-ML, OMEX, COMBINE;CellML;Multicellular modeling <hr>  MorpheusML, https://morpheus.gitlab.io, Multicellular modeling, https://MultiCellML.org, SBML-Spatial, PEtab-MS, https://gitlab.com/fitmulticell/fit | 
| <b>Jürgen Pahle</b> <br/> Heidelberg University | Oct 5 Oct 6 Oct 7 Oct 8 | - <hr> Copasi, CoRC | 
| <b>Lea Seep</b> <br/> University Bonn, IRU-MLS |  | SBML;Ontologies (SBO, KiSAO) <hr> - | 
| <b>Lucian Smith</b> <br/> University of Washington | remotely | SBML;SED-ML, OMEX, COMBINE;Ontologies (SBO, KiSAO);Multicellular modeling <hr> Antimony, Roadrunner, Tellurium, SBML, SED-ML | 
| <b>Luis Fonseca</b> <br/> University of Florida | remotely | SBML;SBGN;CellML;NeuroML;Multicellular modeling;ABM <hr> - | 
| <b>Lukas Buecherl</b> <br/> University of Colorado Boulder |  | SBML;SED-ML, OMEX, COMBINE;SBOL, SBOL visual <hr> - | 
| <b>Lutz Brusch</b> <br/> Technische Universität Dresden, Germany | remotely | Multicellular modeling <hr> Multicellular modeling, https://MultiCellML.org, MorpheusML, https://morpheus.gitlab.io, SBML-Spatial, PEtab-MS, https://gitlab.com/fitmulticell/fit, FAIRSPACE | 
| <b>Maren Philipps</b> <br/> University of Bonn | Oct 5 Oct 6 Oct 7 Oct 8 | - <hr> PEtab, pyPESTO | 
| <b>Mauro Silberberg</b> <br/> Universidad de Buenos Aires, Facultad de Ciencias Exactas y Naturales, Departamento de Física, y CONICET - Universidad de Buenos Aires, Instituto de Física de Buenos Aires (IFIBA). Buenos Aires, Argentina | remotely | SBML;SED-ML, OMEX, COMBINE;CellML;NeuroML <hr> poincaré (github.com/maurosilber/poincare) and SimBio (github.com/hgrecco/simbio), which are Python libraries for definition and simulation of dynamical systems. | 
| <b>Michael Blinov</b> <br/> Center for Cell Analysis and Modeling, UConn Health  | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;SBGN;BioPAX;Ontologies (SBO, KiSAO);Multicellular modeling <hr> VCell, BNGLViz, MolClustPy  | 
| <b>Mustafa Ozen</b> <br/> Altos Labs |  | SED-ML, OMEX, COMBINE;CellML;NeuroML;Multicellular modeling <hr> - | 
| <b>Natasa Miskov-Zivanov</b> <br/> University of Pittsburgh | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;SBGN;BioPAX;SBOL, SBOL visual;CellML;NeuroML;Ontologies (SBO, KiSAO);Multicellular modeling <hr> - | 
| <b>Nilesh Kumar</b> <br/> PhD |  | SBML;SED-ML, OMEX, COMBINE;CellML <hr> - | 
| <b>Olga Krebs</b> <br/> Heidelberg Institute for Theoretical Studies  HITS  |  | SED-ML, OMEX, COMBINE;SBOL, SBOL visual;Ontologies (SBO, KiSAO);BioProtocols <hr> FAIRDOM, LiSyM Cancer, MESI-STRAT, PoLiMeR, deNBI | 
| <b>Paola Vera-Licona</b> <br/> UConn Health | Oct 5 Oct 6 Oct 7 | SBML;SED-ML, OMEX, COMBINE;CellML;Multicellular modeling <hr> NETISCE (NETwork-drIven analysiS of CEllular reprogramming) http://veraliconalab.org/Netisce/index.html | 
| <b>Pedro Mendes</b> <br/> University of Connecticut School of Medicine | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;Ontologies (SBO, KiSAO);Multicellular modeling <hr> - | 
| <b>Prashant Vaidyanathan</b> <br/> Oxford Biomedica | remotely | SBOL, SBOL visual <hr> - | 
| <b>Rahuman Sheriff</b> <br/> European Bioinformatics Institute (EMBL-EBI) | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;SBGN;Ontologies (SBO, KiSAO);Multicellular modeling;FROG <hr> BioModels https://www.ebi.ac.uk/biomodels | 
| <b>Sebastien Moretti</b> <br/> SIB Swiss Institute of Bioinformatics | remotely | SBML;Ontologies (SBO, KiSAO) <hr> - | 
| <b>Sikao Guo</b> <br/> PostDoc | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;BioPAX;CellML;NeuroML;Multicellular modeling <hr> I am currently working on developing and parallelizing NERDSS (Structure-Resolved Reaction-Diffusion Simulation Software). You can find the GitHub page at https://github.com/mjohn218/NERDSS. | 
| <b>T.J. Sego</b> <br/> University of Florida | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;Multicellular modeling <hr> CompuCell3D, Tissue Forge, https://directory.ufhealth.org/sego-t-j | 

<h3><a name="breakouts">Breakout sessions</a></h3>

<b>SBOL Visual</b> Prashant Vaidyanathanm (Oxford Biomedica)	<br>
In these breakout sessions, we will develop into two exciting topics. First, we will explore the SBOL Visual Gallery, a novel initiative aimed at showcasing the diverse applications of SBOL visual diagrams. We will discuss strategies for encouraging the community to share their SBOL visual images and consider ways to link these images to SBOL examples. Additionally, we will brainstorm ideas for promoting these images to a wider audience through social media platforms such as Twitter and LinkedIn. Second, we will commemorate the 10th anniversary of SBOL visual by reflecting on its journey and impact over the past decade. We will discuss the evolution of SBOL visual, its contributions to the field, and its future prospects. Join us for a lively discussion on the past, present, and future of SBOL visual.
<hr>
<b>SBOL Outreach</b> Prashant Vaidyanathan	(Oxford Biomedica)	In these breakout sessions, we will address several key topics related to the SBOL community. First, we will discuss the creation of an SBOL Projects portal or dashboard, a new effort designed to highlight tasks and ongoing projects that the community can contribute to. Second, we will explore the SBOL Release cadence, discussing strategies to ensure that the community can benefit from the latest version of SBOL. We will consider ways to keep our tools and models up-to-date to maximize the benefits for the community. Third, we will discuss outreach efforts, exploring ways to reach a wider SBOL audience, increase accessibility, and boost engagement. Lastly, we will discuss SBOL's involvement with the iGEM community and participants, brainstorming ways to enhance our collaboration and support. Join us for a productive conversation on how to strengthen the SBOL community and increase its impact.
<hr>
<b>SED2: The next generation of Simulation Experiment Design</b>	Eran Agmon	Lucian Smith	UConn Health	<br>
Intended Audience: Researchers, bioinformaticians, software developers, and modelers keen on shaping the future of in silico experiment design standards. Familiarity with SED-ML (https://sed-ml.org/) is advantageous but not essential.<br>

Overview: The evolution of computational biology demands a more adaptable and integrative exchange format. While SED-ML has been instrumental, the emerging needs of the community call for a successor: SED2. This session will focus on the core priorities for SED2: flexibility, the ability to compose different simulation methods with custom annotations, and fostering a computational framework that works with the diverse range of simulation tools within the biological modeling community.<br>

Session Goals:
- SED-ML Reflection: A recap of SED-ML's journey, emphasizing its strengths and areas of improvement.
- Core Priorities for SED2: Deep dive into the need for flexibility and compositionality, and the significance of custom annotations.
- Building a Collaborative Framework: Engage in discussions to establish a common ground where members of the Biosimulations community can converge on SED2's direction.
- Exchange Format and API Exploration: Deliberate on the potential of adopting formats like JSON for SED2 and brainstorm on the design of an intuitive API for seamless interactions with the new format.
<hr>
<b>Standardizing Multicellular Simulation - Bridging Methods and Models</b>	Eran Agmon, UConn Health;	Sego , T.J. (U of Florida); Glazier, James (Indiana University)		<br>
Intended Audience: Researchers, bioinformaticians, software developers, and modelers who are involved in or interested in multicellular simulations across various methodologies. Prior experience with basic concepts in computational biology or multicellular modeling is beneficial but not mandatory.<br>

Overview: Multicellular simulations have become indispensable in understanding complex biological phenomena, from tissue development to disease progression. But the diversity in simulation methods ‚Äì from agent-based models, cellular Potts models, cellular automata, lattice-free models, stochastic particle simulations, etc ‚Äì poses challenges in reproducibility, modularity, reusability, and integration within multi-scale simulation. This session aims to bridge these gaps by focusing on the development of standards and schemas, with special emphasis on multiscale, embedded, and coupled simulation methods. Through a combination of presentations, case studies, and discussions, attendees will gain an understanding of the multicellular simulation landscape, the need for standardization, and the importance of sharing and reusing models.<br>

Session Goals:
- Landscape of Multicellular Simulations: An overview of the various methods employed in multicellular simulations, highlighting their unique features and common challenges.
- The Need for Standards: Discussion on the current gaps and inconsistencies in multicellular simulation methodologies and the implications for research reproducibility and collaboration.
- Designing Schemas for Multicellularity: Collaborative brainstorming on creating robust and flexible schemas that can cater to the diverse range of multicellular simulation methods.
- Integration and Interoperability: Delve into strategies for ensuring that the developed standards promote seamless integration and interoperability among different simulation tools and platforms.
  
<hr>
<b>FROG curation for genome-scale metabolic models</b> Karthik Raman	Rahuman Sheriff and other FROG contributors,	Indian Institute of Technology Madras<br>	
Community standards for consistent reconstruction, FAIR sharing and curation of constraint-based models such as genome-scale metabolic models (GEMs) are crucial to ensure their reproducibility and reliability. We initiated a community effort for a standardised assessment of reproducibility and curation of constraint-based models. Following the discussions at dedicated breakout sessions at HARMONY and COMBINE meetings over the past two
years, we have developed the FROG analysis, an ensemble of analysis of constraint-based
models to test the reproducibility of numerical simulation based on a set of standardized
analyses. FROG analysis encompasses Flux variability analysis (FVA), Reaction deletion
analysis, Objective function calculation, and Gene deletion analysis. We have also
developed a collection of tools that generate FROG reports in a standardized schema to
enable a reliable assessment of model reproducibility based on popular constraint-based
software and a web-service. FROG analysis is currently used in BioModels‚Äô workflow to
curate and build a collection of FAIR and reproducible genome-scale metabolic models.

<hr>
<b>Recent Progress and Future Work Towards Reproducible Stochastic Biological Simulation</b>	T.J. Sego; Rahuman S. Malik Sheriff, University of Florida; European Molecular Biology Laboratory	<br>
Stochastic simulations are commonly used to quantitatively or semi-quantitatively describe the dynamics of biological systems. At various scales and in multiple applications, stochastic simulation better reflects observed biological processes and robustness. Various methods are widely used to incorporate stochasticity into biological simulation, such as the Gillespie stochastic simulation algorithm for systems biology modeling, stochastic Boolean networks for network modeling, and the Cellular Potts model methodology for multicellular modeling. Proving reproducibility of simulation results is critical to establishing the credibility of a model. To this end, BioModels, the largest repository of curated mathematical models, tests and reports the reproducibility of simulation results for all submitted models when possible. A recent study showed that about 50% of the deterministic ordinary differential equation models on BioModels could not be reproduced when applying criteria for reproducibility to the information provided in their associated publication, reflecting a current crisis of reproducibility. Furthermore, there are no well-accepted metrics or standards for reproducing stochastic simulation results, thus perpetuating the crisis of reproducibility for a broad class of biological models. This breakout session aims to establish an accepted framework for testing the reproducibility of stochastic simulations in biological modeling. The session will provide an overview of recent progress towards defining quantitative measures to determine whether stochastic simulation results can be reproduced, and when results have been reproduced. Attendees will discuss current issues to address towards consensus and broad adoption in relevant modeling communities, as well as future work towards reproducibility of stochastic simulation results using multiscale and complex models. 

<hr>
<b>PEtab: current state and future directions</b> Dilan Pathirana	Frank Bergmann	University of Bonn	<br>
PEtab is a standardized file format for specifying parameter estimation problems [1]. The interoperable format is currently supported by 11 different tools [2], enabling users to benefit from standardized parameter estimation across frameworks based in Python, Julia, R, MATLAB, C++, or GUIs.

Although PEtab was initially developed for parameter estimation, recent efforts have extended the format to improve standardization of various adjacent tasks, including: model selection, multi-scale modeling, PKPD and NLME modeling, optimal control, and visualization.

In this breakout session, based on audience interests, we will present introductions to PEtab and its extensions, then discuss current efforts to improve PEtab. People unfamiliar with PEtab are welcome to attend, and might first like to check out the tutorial [3].

[1] ""PEtab‚ÄîInteroperable specification of parameter estimation problems in systems biology"" https://doi.org/10.1371/journal.pcbi.1008646
[2] https://github.com/PEtab-dev/petab#petab-support-in-systems-biology-tools
[3] https://petab.readthedocs.io/en/latest/tutorial.html

<hr>

<b>Annotations for SBML-qual models</b>	John Gennari		University of Washington	<br>
Annotations against standard ontological resources is an important step for model reuse, model merging, and model comprehension. What are the specific needs for annotation of "logical" models (sometimes known as Boolean models)? In this breakout session, we'll look at some example SBML-Qual models, especially gene regulatory networks. The specific augmentations that SBML-qual provides over "plain" SBML means that there are new opportunities to provide specific types of annotations on elements. As with any annotation effort, we will discuss tool support and ways to make annotation semi-automatic or easier for the modeler. We will also discuss how in-line annotations might look within SBML-qual, versus a separate file (per the COMBINE community recommendation).

<h3><a name="tutorials">Tutorials</a></h3>

<b>Biological and Biophysics Simulation in Tissue Forge: Introduction and Guided Simulation Building</b> T.J. Sego		University of Florida	<br>
Tissue Forge is open-source simulation software for interactive particle-based physics, chemistry and biology modeling and simulation. Tissue Forge allows users to create, simulate and explore models and virtual experiments based on soft condensed matter physics at multiple scales, from the molecular to the multicellular, using a simple interface. While Tissue Forge is designed to simplify solving problems in complex subcellular, cellular and tissue biophysics, it supports applications ranging from classic molecular dynamics to agent-based multicellular systems with dynamic populations. Tissue Forge users can build and interact with models and simulations in real-time and change simulation details during execution, or execute simulations off-screen and/or remotely in high-performance computing environments. Tissue Forge provides a growing library of built-in model components along with support for user-specified models during the development and application of custom, agent-based models. Tissue Forge includes an extensive Python API for model and simulation specification via Python scripts, an IPython console and a Jupyter Notebook, as well as C and C++ APIs for integrated applications with other software tools. Tissue Forge supports installations on Windows, Linux and MacOS systems and is available for local installation via conda. This workshop introduces the basic concepts, modeling and simulation features, and some relevant modeling applications of Tissue Forge through guided simulation scripting. Workshop concepts will introduce basic Tissue Forge modeling concepts and simulation features through the development of interactive simulations in Python. Attendees are encouraged, but not required, to code along as the workshop interactively develops and tests simulations in multicellular and biophysics modeling applications. 
<hr>

<b>Bridging the Gap - A Practical Guide to Model Specification Translation in Agent-Based Modeling</b> Juliano Ferrari Gianlupi		UTHSC	<br>
"https://github.com/JulianoGianlupi/pcxml2cc3d
In the world of Agent-Based Modeling (ABM), the quest for cross-platform portability and model reproducibility is a formidable challenge. This tutorial is designed to empower modelers, scientists, and researchers with the knowledge and skills to overcome this challenge using a novel Model Specification Translator.

Agent-Based Modeling has emerged as a vital tool for exploring intricate biological systems, from cancer progression to embryonic development. However, the lack of interoperability and reusability among ABM platforms has raised concerns about model reproducibility. Our tutorial addresses these concerns head-on, introducing a practical and hands-on approach to translate models across different platforms.

During this tutorial, participants will embark on a step-by-step journey through the Model Specification Translation process. We will dive into the essential concepts and methodologies necessary for seamlessly converting models from one platform to another. Whether you're working with CompuCell3D, Tissue Forge, PhysiCell, or any other ABM platform, this tutorial will equip you with the skills needed to ensure your models remain portable and interoperable.

Additionally, we will showcase PhenoCellPy, a Python package that simplifies the creation of cell behavioral patterns. This tool not only enhances model accessibility for biologists but also streamlines the transition from biological concepts to computational implementation.

Join us in this tutorial to explore the significance of cross-platform portability, learn how to overcome the intricacies of model porting, and contribute to the broader ABM community's effort in establishing a universal modeling description standard. By the end of this tutorial, you will be better equipped to advance agent-based modeling, foster model reproducibility, and gain deeper insights into complex biological systems.
<hr>

<b>Mastering Structure-Resolved Reaction-Diffusion Simulations with NERDSS</b> Sikao Guo		Johns Hopkins University <br>
This tutorial (https://sikaoguo22.github.io/NERDSSTutorial/) is intended for researchers, students, and professionals in cellular biology, biophysics, and computational biology who are interested in spatiotemporal reaction-diffusion simulation. NERDSS (https://github.com/mjohn218/NERDSS) is a nonequilibrium reaction-diffusion self-assembly simulator that integrates molecular structures and their processes to understand the dynamics of cellular processes that last for minutes (https://doi.org/10.1016/j.bpj.2020.05.002). It allows users to build a reaction-diffusion model based on actual molecular structures, which enhances the model's accuracy and captures the complexities of multisubunit complexes and their reversible formation. Several case studies have employed NERDSS, such as the formation and spontaneous disassembly of large clathrin lattices (https://doi.org/10.1371/journal.pcbi.1009969), the dynamic behavior of the HIV Gag lattice in virions (https://doi.org/10.7554/eLife.84881), and understanding the temporal influence of cofactors in retroviral Gag lattice assembly (https://doi.org/10.1016/j.bpj.2023.06.021). During the session, we will explore these applications, demonstrating the software's versatility in handling diverse cellular processes. The session will cover core principles of structure-resolved reaction-diffusion, emphasizing its role in cellular biology. We will attempt to build a coarse-grained structure from the real protein structure from the pdb database. Then, we will learn how to set up a model, run the model with NERDSS, analyze and visualize the model outcomes with io_nerdss (https://github.com/mjohn218/io_nerdss) and OVITO.
<hr>

<b>Tutorial on biological modeling with PySB</b> Mustafa Ozen	Ryan Spangler, Carlos F. Lopez	Altos Labs <br>
PySB (Python Systems Biology) is a powerful and versatile biological modeling tool that has gained prominence in systems biology. It provides a Python-based programming platform with a rule-based framework for constructing dynamic models of sophisticated biochemical systems, enabling researchers to simulate and analyze complex cellular processes. This unique tool empowers researchers to modularly define molecular interactions and transformations, facilitating the representation of a wide array of biological processes in a simple, interpretable way. PySB accommodates both stochastic and deterministic simulation methods, providing a comprehensive view of system behavior. Its seamless integration with various model calibration, analysis, and visualization libraries further assists researchers in interpreting simulation results effectively. In this tutorial session, we aim to walk the attendants through the foundations of PySB, show them how it works, and provide them with hands-on experience.  

Relevant Resources:
PySB Paper: https://www.embopress.org/doi/full/10.1038/msb.2013.1
PySB Website: https://pysb.org/
PySB GitHub: https://github.com/pysb/pysb
PySB Tutorial: https://pysb.readthedocs.io/en/stable/tutorial.html

<hr>
<b>Introducing poincar√© and SimBio</b> Mauro Silberberg		Universidad de Buenos Aires, Facultad de Ciencias Exactas y Naturales, Departamento de F√≠sica, y CONICET - Universidad de Buenos Aires, Instituto de F√≠sica de Buenos Aires (IFIBA). Buenos Aires, Argentina	
<br>
Poincar√© is a new Python-based library to define and simulate dynamical systems. Trying not to reinvent (much of) the wheel, it uses modern Python syntax to define and compose models in a compact way. This allows us to take advantage of the huge investment on tooling from the broader Python community such as static type checkers, code linters and formatters, autocomplete and refactoring features from IDEs, among others.

Using standard libraries from the PyData ecosystem, by default, poincar√© compiles into a first-order ODE system using NumPy arrays, and uses solvers from SciPy. But it also provides different backends such as Numba, which compiles just-in-time to LLVM code, providing a significant speed boost, or JAX, which provides autodifferentiation tools targeted for ML. Additionally, it supports units using the Pint library.

SimBio is built on top of poincar√©, adding some components for reaction-based models used in systems biology. It provides some predefined building blocks for the most common reactions, but allows easily to create your own. Finally, it implements an importer and exporter to SBML, allowing to interexchange models with the COMBINE community. We hope that it is simple enough for beginners, but powerful for power-users with the possibility to extend and compose with the large Python ecosystem.
<hr>

<b>DySE: Dynamic System Explanation framework</b>
Tang, Difei (University of Pittsbrugh); Miskov-Zivanov Natasa (University of Pittsburgh)	University of Pittsburgh	<br>
In this demo, we will showcase our framework, DySE (Dynamic System Explanation), that includes tools for model simulation, model extension, interaction classification, interaction filtering, model checking, and sensitivity analysis. The rapid proliferation of data generated by experiments studying biological systems poses a considerable challenge. This overflow of information is spread across an array of publishing platforms, making it increasingly difficult to manually analyze all available data. This underscores the necessity for automated methods that can retrieve and connect relevant pieces of this voluminous knowledge. Such methods are crucial for understanding, explaining, and predicting the behavior of these complex systems.
 
To tackle this challenge, DySE integrates machine reading, automated model assembly, and computational analysis to enhance understanding and explanation of complex systems. The toolset is conveniently accessible via a user-friendly graphical interface (GUI).
 
FLUTE utilizes existing databases to evaluate confidence and trustworthiness of given biochemical interactions (https://tinyurl.com/flutedoc). VIOLIN classifies large sets of interactions with respect to a given model. The interactions are classified into four main categories, corroborations, contradictions, extensions, and flagged, and several subcategories within main ones (https://tinyurl.com/violindoc). CLARINET (see https://tinyurl.com/clarinetdoc and link to binder notebook on main page) and ACCORDION (see https://tinyurl.com/accordiondoc and link to binder notebook on main page) automatically expand and recommend models based on selected desired model properties.
 
In conjunction with these front-end tools, we've also developed back-end tools within the DySE framework. DiSH is a stochastic simulator offering versatile simulation schemes and timing options (https://tinyurl.com/dishjupyter). PIANO provides comprehensive sensitivity analysis for the entire model, and allows for identifying most influential pathways and suggesting interventions (https://tinyurl.com/pianojupyter). Additionally, we've introduced a unified format compatible with the tools mentioned above: BioRECIPE (https://tinyurl.com/biorecipe), seamlessly translating to and from widely used synthetic biology modeling languages.
 
All of these tools can be used either independently or in combination. For instance, synthetic biologists can rely on it to ensure the reliability of interactions in designing synthetic biological systems. Bioinformaticians can efficiently filter and prioritize data, while computational modelers benefit from model extensions to always get model up-to-date. Professionals in biotech industries utilize sensitivity analysis for optimizing therapies. Additionally, even educators would find it valuable when using simulation software.
<hr>

<b> Novel advances in the automation of knowledge selection and model assembly</b> 	Natasa Miskov-Zivanov, 	Ahmed, Yasmine; Zhou, Gaoxiang;	University of Pittsburgh <br>
Creating computational models of complicated systems, including intracellular and intercellular bionetworks, is a time and labor-intensive task which is often limited by the knowledge and experience of Pathway database modelers. This has naturally led to the emergence of the idea of automating the process of building new/extending existing models, which could have a significant potential in enabling rapid, consistent, comprehensive and robust analysis of complicated systems. Inspired by this idea, we propose in this work different novel approaches namely ACCORDION (ACCelerating and Optimizing model RecommenDatIONs) and CLARINET (CLARifying NETworks) for expanding models using the information extracted from literature by machine reading engines. Our proposed approaches combine machine reading with clustering, and graph theoretical analysis to create an automated framework for efficient model assembly. Furthermore, by automatically extending models with the information published in literature, our proposed methods allow for collecting the existing information in a consistent and comprehensive way. This, in turn, facilitates information reuse, data reproducibility, and replacing hundreds/thousands of manual experiments, thereby reducing the time needed for the advancement of knowledge. To evaluate ACCORDION1 and CLARINET, we compare their outcomes with three previously published manually created models namely naive T cell differentiation model, T cell large granular lymphocyte leukemia model and pancreatic cancer cell model. Besides demonstrating automated reconstruction of a model that was previously built manually, our tools can assemble multiple models that satisfy desired system properties. As such, they replace large number of tedious or even impractical manual experiments and guide alternative hypotheses and interventions in biological systems.
A GitHub page, ReadtheDocs and Jupyter notebook are available for ACCORDION http://www.nmzlab.pitt.edu/accordion and CLARINET http://www.nmzlab.pitt.edu/clarinet.
<hr>
