---
title: COMBINE 2023

# Is this the primary user of the site?
superuser: false

conf_date: 5-8, October 2023

conf_location: Farmington, CT

important_dates:
- name: Breakouts, talks and tutorials submission deadline
  date: September 1, 2023
- name: Lightning talk and poster submission deadline
  date: September 15, 2023
- name: Notification of acceptance of breakouts and tutorials
  date: September 7, 2023
- name: DEADLINE for student travel support applications
  date: September 1, 2023

important_links:
#- name: Agenda
#  link: 
- name: Registration
  link: https://forms.gle/7pbiYps84yUvEbsY9
- name: Break-outs and tutorials submission
  link: https://forms.gle/i1ENK9NicXBshhUM7
- name: Talks, lightning talks, and poster submission
  link: https://forms.gle/3ej8xnxQotzz6ZV99
- name: Application for student travel support
  link: https://forms.gle/whr1MusnSEFZD3vy8

information:
- name: Registration
  description: Registration for the meeting is free. Please register at the link above as soon as possible. This will help us plan the schedule and match your interests to the timing of the breakouts, etc. Note, only registered attendees will be sent information related to the meeting.
  link: https://forms.gle/7pbiYps84yUvEbsY9
- name: Call for Breakout Sessions and Tutorials
  description: All attendees can suggest breakout sessions for hacking and/or detailed discussions of certain aspects of one or several of the COMBINE standard(s), metadata and semantic annotations (format-specific or overarching), application and implementations of the COMBINE standards, or any other topic relevant for the COMBINE community. The topics for those breakout sessions, and the time slots which would suit their communities can be submitted via the link above. Note, breakout session organisers will be responsible for creating and hosting their own online sessions, if required.
  link: https://forms.gle/i1ENK9NicXBshhUM7
- name: Call for Talks, Lightning Talks and Posters
  description: Requests for a talk (40 minutes), lightning talk (5 min max.) and/or poster can be submitted via the link above. Please use several forms if you want to submit abstracts on different topics. The submission deadline is outlined above. Talks will take place during the community session and posters will be displayed throughout the meeting.
  link: https://forms.gle/3ej8xnxQotzz6ZV99


topics:
- Data exchange, pipelines and model standards for systems and synthetic biology
- Visualization and graphical notation standards for systems and synthetic biology
- Standards for sharing and analysing biological pathway data
- Standards for computational biological models and modelling support
- Metadata description and model annotation in COMBINE standard formats
- Implementation of COMBINE standards in tools, databases and other resources
- Integrated model and data management for systems and synthetic biology
- Standardization of Artificial Intelligence approaches in biological modelling
- Emerging standardization needs and multicellular modeling
- Community aspects of COMBINE

# Organizational groups that you belong to (for People widget)
#   Set this to `[]` or comment out if you are not using People widget.
user_groups:
- Events

---
<img src="/images/combine2023/combine23-hartford.png" alt="demo" class="img-responsive">
<font color = red>NEWS: list of attendees as of September 9 is posted below </font>
<br>

The "**Computational Modeling in Biology**" Network (**COMBINE**) is an initiative to coordinate the development of the various community standards and formats in systems biology and related fields. **COMBINE 2023** will be a workshop-style event hosted at the [Center for Cell Analysis and Modeling](https://health.uconn.edu/cell-analysis-modeling/) at the [University of Connecticut School of Medicine](https://medicine.uconn.edu/), Farmington, CT, USA. The meeting will be held in October 2023, closely aligned with the dates of the [**International Conference on Systems Biology (ICSB)**](https://icsb2023.bioscience-ct.net/) October 8-12 in Hartford, CT. Tutorials at the last day (Sunday October 8th) will take place at the ICSB 2023 venue - in the convention center. The meeting days will include talks about the COMBINE standards and associated or related standardization efforts, presentations of tools using these standards, breakout sessions for detailed discussions as well as tutorials. There are no dedicated poster sessions, but paricipants are encouraged to bring posters - poster boards will be provided next to meeting places. Some time each day will be left for community discussion and wrap-ups of breakouts and advertisements for following breakouts. It will be primarily an in-person meeting, with individual breakout sessions responsible for enabling remote participation as needed.

Local organizers are Michael Blinov (blinov@uchc.edu) and Ion Moraru (moraru@uchc.edu). 

<font color = red>Students may apply for a travel support from NSF!!! Link is on the LEFT! </font>

<h3>A tentative schedule</h3>

<img src="/images/combine2023/schedule.png" width="100%" alt="schedule.png">Note that many events are scheduled somewhat spontaneously at these events; keep an eye out here or on the COMBINE slack for last-minute changes and additions.

<h3>Workshop Location</h3>

COMBINE 2023 takes place at the [Center for Cell Analysis and Modeling (CCAM)](https://health.uconn.edu/cell-analysis-modeling/) in Farmington, CT. **COMBINE 2023** will take place in the Cell and Genome Sciences building, at [400 Farmington Ave, Farmington, CT 06119](https://goo.gl/maps/Ms25yakj7NU7dFsv5). If the doors are locked, please knock, email blinov@uchc.edu or moraru@uchc.edu, or ping us on the [COMBINE slack channel](https://combine-workspace.slack.com).

<img src="/images/combine2023/ccam3.png" width="100%" alt="ccam3.png">

<h3>Arrival and Transportation</h3>

The closest airport is the [Bradley International Airport (BDL)](https://bradleyairport.com/). The recommended transportation is to take Uber/Lyft to 400 Farmington Ave, Farmington, CT. It takes about 25 minutes and costs about $40. You may rent a car at BDL - all parking at the workshop location and hotel are free. A less expensive but long travel from BDL is to take Bradley Flyer Bus (#30) from BDL to Central Row North Side at Old State House Station, and then take a [bus 66T](https://goo.gl/maps/9BpUPEqBUb5Zgrnm7) from Main St & Asylum St to 400 Farmington Ave. It takes 1 to 3 hours depending on schedules. 

<h3>Accomodations and meals</h3>

The closest hotel to the CCAM is the [Homewood Suites by Hilton Hartford-Farmington](https://www.hilton.com/en/hotels/hfdfmhw-homewood-suites-hartford-farmington/) at [2 Farm Glen Boulevard, Farmington, Connecticut, 06032](https://goo.gl/maps/jMK2dqEYqKSDf7vC7); it's [7 minutes walk](https://goo.gl/maps/D5j5SQnSLiRQFaKx9) between there and the venue. A University rate of $154/night will be provided upon request. There are more hotels in the area, but any other hotel will require a car.

The hotel serves hot breakfast. There is a [Butchers and Bakers restaurant](https://butchersandbakers.com/) 15 minutes walk, but no food stores within walking distance. We will serve food at the venue, depending on the sponsors it may be free or for a nominal fee.

<h3>Attendees</h3>

38 registered participants as of September 9
| Name <hr> Affiliation  | Attendance | Interested <hr>  Projects |
|---|---|---|
| <b>Amir Mahari</b> <br/> University of Arkansas | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SBGN;Multicellular modeling <hr> I focus on the modeling and simulation of intercellular signaling pathways within melanoma cancer cells. This study holds significance due to the intricate nature of signaling cascades that regulate various cellular processes in melanoma progression. To facilitate this investigation, I employ PySB (Python Systems Biology), a powerful computational framework designed for creating, simulating, and analyzing mathematical models of biochemical systems. PySB's modular and user-friendly nature enables me to construct complex models of intercellular signaling, incorporating factors such as ligand-receptor interactions, protein phosphorylation, and gene expression. Through PySB's simulation capabilities, I can dynamically analyze the behavior of these pathways under different conditions, gaining insights into the underlying mechanisms that drive melanoma growth and metastasis. Ultimately, this research using PySB contributes to an enhanced understanding of melanoma biology, potentially paving the way for novel therapeutic interventions and personalized treatment strategies in combating this aggressive form of cancer. | 
| <b>Augustin Luna</b> <br/> Harvard Medical School | remotely | SBGN;BioPAX <hr> https://www.pathwaycommons.org/; https://biofactoid.org/ | 
| <b>Bharat Mishra</b> <br/> PhD |  | SBML;SED-ML, OMEX, COMBINE;CellML;Ontologies (SBO, KiSAO);Multicellular modeling <hr> https://www.uab.edu/cores/ircp/bds | 
| <b>Chris Myers</b> <br/> University of Colorado Boulder | Oct 5 Oct 6 Oct 7 | SBML;SED-ML, OMEX, COMBINE;SBOL, SBOL visual <hr> SynBioHub/SynBioSuite/iBioSim | 
| <b>Dagmar Waltemath</b> <br/> University Medicine Greifswald | remotely | SED-ML, OMEX, COMBINE;Ontologies (SBO, KiSAO) <hr> - | 
| <b>Daniel Ajuzie</b> <br/> undergraduate | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;SBOL, SBOL visual;CellML;Ontologies (SBO, KiSAO);Multicellular modeling <hr> Ensemble Modeling; Model Optimization and Calibration | 
| <b>Daniel Ajuzie</b> <br/> undergraduate | Oct 5 Oct 6 Oct 7 | SBML;SED-ML, OMEX, COMBINE;CellML;NeuroML <hr> Multi-phenotype Optimization for Dynamic Modeling | 
| <b>Diego Jahn</b> <br/> TUD Dresden University of Technology, Center for Information Services and High Performance Computing (ZIH) | Oct 5 Oct 6 | Multicellular modeling <hr> MorpheusML, MorpheusML Model Repository (https://morpheus.gitlab.io), Multicellular Modeling | 
| <b>Egils Stalidzans</b> <br/> PhD |  | SBML;SBOL, SBOL visual <hr> ODE-based kinetic modeling, constraint based stoichiometric modeling | 
| <b>Elebeoba May</b> <br/> University of Wisconsin-Madison |  | SBML;Multicellular modeling;Visualization <hr> biochemical/bionetwork ODE models (metabolism, signal transduction, gene networks), multiscale models of host-pathogen systems, multiscale models of microbial communities, modeling synthetic bio systems | 
| <b>Fengkai Zhang</b> <br/> NIH |  | SBML;SED-ML, OMEX, COMBINE;SBGN <hr> - | 
| <b>Frank T. Bergmann</b> <br/> BioQUANT, Heidelberg University | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;SBGN <hr> COPASI (https://copasi.org), basico (https://basico.readthedocs.io/), libSBML / libSEDML / libCombine | 
| <b>Gaoxiang Zhou</b> <br/> University of Pittsburgh | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;BioPAX;SBOL, SBOL visual;CellML;Multicellular modeling <hr> https://www.nmzlab.pitt.edu/people/gaoxiang-zhou, https://github.com/pitt-miskov-zivanov-lab, https://melody-biorecipe.readthedocs.io | 
| <b>Gerhard Mayer</b> <br/> HITS (Heidelberg Institute for Theoretical Studies) gGmbH, Heidelberg | remotely | SED-ML, OMEX, COMBINE;Multicellular modeling <hr> EDITH (Ecosystem Digital Twins in Healthcare); https://www.edith-csa.eu | 
| <b>James A. Glazier</b> <br/> Indiana University | Oct 5 Oct 6 Oct 7 Oct 8 | Multicellular modeling <hr> CompuCell3D (https://compucell3d.org/), Virtual Cornea, IMAG/MSM Working Group on Multiscale Modeling and Viral Pandemics (https://www.imagwiki.nibib.nih.gov/working-groups/multiscale-modeling-and-viral-pandemics) | 
| <b>John Gennari</b> <br/> University of Washington | Oct 5 Oct 6 Oct 7 | SBML;SED-ML, OMEX, COMBINE;BioPAX;CellML;Ontologies (SBO, KiSAO) <hr> Center for Reproducible Biomedical Modeling | 
| <b>Juliano Ferrari Gianlupi</b> <br/> Postdoctoral Scholar, UTHSC |  | SBML;CellML;Multicellular modeling <hr> PhenoCellPy https://www.biorxiv.org/content/10.1101/2023.04.12.535625v2.abstract; Translating PhysiCell specification into CompuCell3D simulation https://github.com/JulianoGianlupi/pcxml2cc3d | 
| <b>Jörn Starruß</b> <br/> Technische Universität Dresden, Germany | remotely | SBML;SED-ML, OMEX, COMBINE;CellML;Multicellular modeling <hr>  MorpheusML, https://morpheus.gitlab.io, Multicellular modeling, https://MultiCellML.org, SBML-Spatial, PEtab-MS, https://gitlab.com/fitmulticell/fit | 
| <b>Jürgen Pahle</b> <br/> Heidelberg University | Oct 5 Oct 6 Oct 7 Oct 8 | - <hr> Copasi, CoRC | 
| <b>Lea Seep</b> <br/> University Bonn, IRU-MLS |  | SBML;Ontologies (SBO, KiSAO) <hr> - | 
| <b>Luis Fonseca</b> <br/> University of Florida | remotely | SBML;SBGN;CellML;NeuroML;Multicellular modeling;ABM <hr> - | 
| <b>Lukas Buecherl</b> <br/> University of Colorado Boulder |  | SBML;SED-ML, OMEX, COMBINE;SBOL, SBOL visual <hr> - | 
| <b>Lutz Brusch</b> <br/> Technische Universität Dresden, Germany |  | Multicellular modeling <hr> Multicellular modeling, https://MultiCellML.org, MorpheusML, https://morpheus.gitlab.io, SBML-Spatial, PEtab-MS, https://gitlab.com/fitmulticell/fit, FAIRSPACE | 
| <b>Maren Philipps</b> <br/> University of Bonn | Oct 5 Oct 6 Oct 7 Oct 8 | - <hr> PEtab, pyPESTO | 
| <b>Mauro Silberberg</b> <br/> Universidad de Buenos Aires, Facultad de Ciencias Exactas y Naturales, Departamento de Física, y CONICET - Universidad de Buenos Aires, Instituto de Física de Buenos Aires (IFIBA). Buenos Aires, Argentina | remotely | SBML;SED-ML, OMEX, COMBINE;CellML;NeuroML <hr> poincaré (github.com/maurosilber/poincare) and SimBio (github.com/hgrecco/simbio), which are Python libraries for definition and simulation of dynamical systems. | 
| <b>Michael Blinov</b> <br/> Center for Cell Analysis and Modeling, UConn Health,  | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;SBGN;BioPAX;Ontologies (SBO, KiSAO);Multicellular modeling <hr> VCell, BNGLViz, MolClustPy,  | 
| <b>Mustafa Ozen</b> <br/> Altos Labs |  | SED-ML, OMEX, COMBINE;CellML;NeuroML;Multicellular modeling <hr> - | 
| <b>Nilesh Kumar</b> <br/> PhD |  | SBML;SED-ML, OMEX, COMBINE;CellML <hr> - | 
| <b>Olga Krebs</b> <br/> Heidelberg Institute for Theoretical Studies  HITS  |  | SED-ML, OMEX, COMBINE;SBOL, SBOL visual;Ontologies (SBO, KiSAO);BioProtocols <hr> FAIRDOM, LiSyM Cancer, MESI-STRAT, PoLiMeR, deNBI | 
| <b>Rahuman Sheriff</b> <br/> European Bioinformatics Institute (EMBL-EBI) | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;SED-ML, OMEX, COMBINE;SBGN;Ontologies (SBO, KiSAO);Multicellular modeling;FROG <hr> BioModels https://www.ebi.ac.uk/biomodels | 
| <b>Sebastien Moretti</b> <br/> SIB Swiss Institute of Bioinformatics | remotely | SBML;Ontologies (SBO, KiSAO) <hr> - | 
| <b>Sikao Guo</b> <br/> PostDoc | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;BioPAX;CellML;NeuroML;Multicellular modeling <hr> I am currently working on developing and parallelizing NERDSS (Structure-Resolved Reaction-Diffusion Simulation Software). You can find the GitHub page at https://github.com/mjohn218/NERDSS. | 
| <b>T.J. Sego</b> <br/> University of Florida | Oct 5 Oct 6 Oct 7 Oct 8 | SBML;Multicellular modeling <hr> CompuCell3D, Tissue Forge | 
