<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 4.8.0 for Hugo"><meta name=author content="COMBINE"><meta name=description content><link rel=alternate hreflang=en-us href=/events/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#2962ff"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark disabled><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.css integrity="sha512-1xoFisiGdy9nvho8EgXuXvnpR5GAMSjFwp40gSRE3NwdUdIMIKuPa7bqoUhLD0O/5tPNhteAsE5XyyMi5reQVA==" crossorigin=anonymous><script src=https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin=anonymous async></script><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=/css/wowchemy.css><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/images/icon_hu506751a9b1968827413e70b30c336728_17544_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/images/icon_hu506751a9b1968827413e70b30c336728_17544_192x192_fill_lanczos_center_2.png><link rel=canonical href=/events/><meta property="twitter:card" content="summary"><meta property="og:site_name" content="Combine: Coordinating standards for modeling in Biology"><meta property="og:url" content="/events/"><meta property="og:title" content="Events | Combine: Coordinating standards for modeling in Biology"><meta property="og:description" content><meta property="og:image" content="/images/logo.svg"><meta property="twitter:image" content="/images/logo.svg"><meta property="og:locale" content="en-us"><title>Events | Combine: Coordinating standards for modeling in Biology</title></head><body id=top data-spy=scroll data-offset=70 data-target=#navbar-main><script>window.wcDarkLightEnabled=true;</script><script>const isSiteThemeDark=false;</script><script src=/js/load-theme.js></script><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/><img src=/images/logo.svg alt="Combine: Coordinating standards for modeling in Biology"></a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/><img src=/images/logo.svg alt="Combine: Coordinating standards for modeling in Biology"></a></div><div class="navbar-collapse main-menu-item collapse justify-content-end" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/people><span>People</span></a></li><li class=nav-item><a class=nav-link href=/standards><span>Standards</span></a></li><li class=nav-item><a class="nav-link active" href=/events data-target=[]><span>Events</span></a></li><li class=nav-item><a class=nav-link href=/documents><span>Documents</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav><span class="js-widget-page d-none"></span><section id=current class="home-section wg-event-full"><div class=container><div class=row><div class="col-12 col-lg-4"><h1 style=color:#1f646b>Harmony 2023</h1></div></div><div class=row><div class="col-12 col-lg-4"><h2 style=color:#8dc742>Conference Date</h2><b>24-27, April 2023</b><h2 style=color:#8dc742>Conference Location</h2><b>Seattle, WA</b><h2 style=color:#8dc742>Important Dates</h2><div>Breakout submission deadline (for initial scheduling)</div><b>April 2, 2023</b><p></p><div>Lightning talk and poster submission deadline</div><b>April 2, 2023</b><p></p><div>Notification of acceptance of breakouts and abstracts</div><b>April 7, 2023</b><p></p><div>HARMONY Workshop</div><b>April 24 to 27, 2023</b><p></p><h2 style=color:#8dc742>Important Links</h2><div class=description><a href=https://combine-org.github.io/events/ style=color:#1f646b>Agenda</a></div><div class=description><a href=https://forms.gle/Nqec2JdyRj12HobK9 style=color:#1f646b>Registration</a></div><div class=description><a href=https://forms.gle/rYZhqbekKLs1RY3B7 style=color:#1f646b>Breakout submission</a></div><div class=description><a href=https://forms.gle/hcpWJrKCtFQDuPwR6 style=color:#1f646b>Lightning talk and poster submission</a></div></div><div class="col-12 col-lg-8"><p></p><p>The Computational Modeling in Biology Network (COMBINE) is an initiative to coordinate the development of the various community standards and formats in systems biology, synthetic biology and related fields. HARMONY is a codefest-type meeting, with a focus on development of the standards, interoperability and infrastructure. There are generally not many general discussions or oral presentations during HARMONY; instead, the time is devoted to allowing hands-on hacking and interaction between people focused on practical development of software and standards.</p><p>HARMONY 2023 will be hosted by Herbert Sauro and the Center for Reproducible Biomedical Modeling at the University of Washington, Seattle, WA. It is going to be organized primarily as an in-person meeting, with individual breakout sessions responsible for enabling remote participation as needed.</p><p>Most of each day will be scheduled by the communities as breakouts. In addition, we will have some time each day for community discussion and wrap-ups of breakouts and advertisements for following breakouts.</p><h3>Schedule</h3><p>Here&rsquo;s the <a href="https://docs.google.com/spreadsheets/d/1Jjs22pMGjJZfeT7tp8btoJxQLXnCdWysEIKn26U72_4/edit#gid=0" target=_blank rel=noopener>full (editable) schedule</a>. Note that many events are scheduled somewhat spontaneously at these events; keep an eye out here or on the COMBINE discord for last-minute changes and additions.</p><h3>Workshop Location Details</h3><p>HARMONY 2023 takes place at the University of Washington (UW, and commonly called &ldquo;U-Dub&rdquo;) in Seattle, Washington. HARMONY 2023 will take place in the William H. Foege Bioengineering building, at <a href=https://goo.gl/maps/rEGPZXcLkiHBcwBa7 target=_blank rel=noopener>3720 15th Ave NE, Seattle, WA 98195</a>. The building is locked during the day, but will be open the half-hour before the meeting begins. If you arrive later than that, call the department reception at 206-685-2000, or email any of the organizers, or ping us on the <a href=combine-workspace.slack.com>COMBINE slack channel</a>.</p><img src="/images/harmony2023/foege and map.png" width=95% alt="Foege building and map"><p>The doors will be locked, so please knock (there&rsquo;s lots of glass), call the Department front desk at (206) 685-2000 or email lpsmith@uw or <a href=mailto:bioeasst@uw.edu>bioeasst@uw.edu</a> or ping us on the COMBINE Slack!</p><h3>Arrival and Transportation</h3><p>From the Seatac Airport, take the <a href=https://www.soundtransit.org/ride-with-us/stations/link-light-rail-stations target=_blank rel=noopener>Link Light Rail</a> north from the airport, and exit at the University of Washington station (NOT Univeristy <em>Street</em> Station; that&rsquo;s a downtown stop). It&rsquo;s a relatively short walk <a href=https://goo.gl/maps/82vspAtwTpG439xN7 target=_blank rel=noopener>from there to the Bioengineering building</a>. The nearest hotel is <a href=https://goo.gl/maps/6TMqczLss91J3TZL7 target=_blank rel=noopener>slightly further</a>; a little over one mile, so you might want to bus/taxi depending on how much luggage you have. If you get an <a href=https://www.soundtransit.org/ride-with-us/how-to-pay target=_blank rel=noopener>ORCA card</a>, you can use that on any of the buses or other light rail trips you might take during your stay. There&rsquo;s also a <a href=https://kingcounty.gov/en/depts/metro/fares-and-payment/ways-to-pay/transit-go-ticket target=_blank rel=noopener>&lsquo;Transit GO Ticket&rsquo;</a> app you should be able to use from your phone (untested by me, but it seems reasonable?).</p><h3>Hotels</h3><p>The closest hotel to the Bioengineering department is the <a href=https://www.staypineapple.com/university-inn-seattle-wa target=_blank rel=noopener>Univeristy Inn</a> at <a href=https://goo.gl/maps/eZ7bJMLNi5hBWr2H6 target=_blank rel=noopener>4140 Roosevelt Way NE, Seattle, WA 98105</a>; it&rsquo;s about a <a href=https://goo.gl/maps/bdWK28hoyLREGFff8 target=_blank rel=noopener>half mile walk</a> between there and the venue. About a block further (and inexplicably slightly more expensive) is the <a href=https://www.staypineapple.com/watertown-hotel-seattle-wa target=_blank rel=noopener>Watertown Hotel</a>, at <a href=https://goo.gl/maps/3k6jrAKmF9JocTo66 target=_blank rel=noopener>4242 Roosevelt Way NE, Seattle, WA 98105</a>. Any hotel anywhere along the <a href=https://www.soundtransit.org/ride-with-us/stations/link-light-rail-stations target=_blank rel=noopener>Link Light Rail</a> line will be easy to get to/from as well, as the Univeristy of Washington station is just a half mile away from the venue.</p><h3>Attendees</h3><p>The following is the list of known attendees as of 24/04/2023 23:58:54 UTC (72 total registered, 63 listed below)</p><table><thead><tr><th style=text-align:left>Name</th><th style=text-align:left>Affiliation</th><th style=text-align:left>Attendance</th></tr></thead><tbody><tr><td style=text-align:left>Eran Agmon</td><td style=text-align:left>UConn Health</td><td style=text-align:left>In person<br>Multicellular modeling, SBML, SED-ML<br>BioSimulators: <a href=https://biosimulators.org>https://biosimulators.org</a>, Vivarium: <a href=https://vivarium-collective.github.io>https://vivarium-collective.github.io</a></td></tr><tr><td style=text-align:left>Azka Ahmed</td><td style=text-align:left>UW Madison</td><td style=text-align:left>Remotely (virtually)<br>BioPAX, CellML, COMBINE Archive, Multicellular modeling</td></tr><tr><td style=text-align:left>Daniel Ajuzie</td><td style=text-align:left>Wisconsin Institute for Discovery</td><td style=text-align:left>Remotely (virtually)<br>BioPAX, CellML, COMBINE Archive, Multicellular modeling, SBML, SED-ML<br>Our lab develops models at multiple scales ranging from gene network, metabolic network, signal transduction to multicellular models. We model individual cells (bacteria and mammalian) and interactions between cells (microbial and host-microbe).</td></tr><tr><td style=text-align:left>Steve Andrews</td><td style=text-align:left>UW</td><td style=text-align:left>In person<br><br>Spatial modeling, including particularly with the Smoldyn package.</td></tr><tr><td style=text-align:left>Mihail Anton</td><td style=text-align:left>NBIS, SciLifeLab, Chalmers University of Technology</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, OMEX Metadata, SBGN, SBML<br>standard-GEM</td></tr><tr><td style=text-align:left>Bryan Bartley</td><td style=text-align:left>Raytheon BBN</td><td style=text-align:left>In person<br>SBOL and SBOL Visual</td></tr><tr><td style=text-align:left>Jacob Beal</td><td style=text-align:left>Raytheon BBN</td><td style=text-align:left>In person<br>SBOL and SBOL Visual</td></tr><tr><td style=text-align:left>Frank Bergmann</td><td style=text-align:left>Heidelberg University</td><td style=text-align:left>In person<br>COMBINE Archive, SBGN, SBML, SED-ML<br>COPASI (copasi.org), basico (basico.rtfd.io), libsbml, libsedml, libcombine</td></tr><tr><td style=text-align:left>Lutz Brusch</td><td style=text-align:left>Technische Universität Dresden, Germany</td><td style=text-align:left>Remotely (virtually)<br>Multicellular modeling, SBML<br>COMBINE Archive and OMEX, PEtab, MorpheusML, SBML-Spatial, FAIRSPACE, <a href=https://www.MultiCellML.org>https://www.MultiCellML.org</a></td></tr><tr><td style=text-align:left>Lukas Buecherl</td><td style=text-align:left>University of Colorado Boulder</td><td style=text-align:left>In person<br>COMBINE Archive, SBML, SBOL and SBOL Visual, SED-ML<br>Genetic Design Automation</td></tr><tr><td style=text-align:left>Kevin Chen</td><td style=text-align:left>University of Washington</td><td style=text-align:left>In person<br>BioPAX, CellML, COMBINE Archive, NeuroML, OMEX Metadata, SBML, SED-ML</td></tr><tr><td style=text-align:left>Rafael Costa</td><td style=text-align:left>FCT-NOVA, NOVA School of Science and Technology</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, SBML</td></tr><tr><td style=text-align:left>Alan Garny</td><td style=text-align:left>University of Auckland</td><td style=text-align:left>In person<br>CellML, COMBINE Archive, Multicellular modeling, OMEX Metadata, SED-ML<br>libOpenCOR and OpenCOR</td></tr><tr><td style=text-align:left>Tom Gebhardt</td><td style=text-align:left>University of Greifswald</td><td style=text-align:left>Remotely (virtually)<br>CellML, COMBINE Archive, SBGN, SBML</td></tr><tr><td style=text-align:left>John Gennari</td><td style=text-align:left>U of Washington</td><td style=text-align:left>In person<br>BioPAX, CellML, OMEX Metadata, SBML, SBOL and SBOL Visual, SED-ML</td></tr><tr><td style=text-align:left>James Glazier</td><td style=text-align:left>Indiana University</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, Multicellular modeling, SBML<br>Replication and annotation of multicellular models</td></tr><tr><td style=text-align:left>Issa Gnasse</td><td style=text-align:left>Pasteur Institute of Dakar</td><td style=text-align:left>In person<br>Multicellular modeling, OMEX Metadata<br>Application of Machine learning to find early biomarker in cervical cancer.</td></tr><tr><td style=text-align:left>Misirli Goksel</td><td style=text-align:left>Keele University</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, OMEX Metadata, SBML, SBOL and SBOL Visual</td></tr><tr><td style=text-align:left>Jenn Hadlock</td><td style=text-align:left>Insitute for Systems Biology</td><td style=text-align:left>Remotely (virtually)<br>BioPAX, OMEX Metadata, SBML<br>NCATS Biomedical Translator</td></tr><tr><td style=text-align:left>Joseph Hellerstein</td><td style=text-align:left>eScience Institute, University of Washington</td><td style=text-align:left>In person<br>COMBINE Archive, SBML<br>VSCode-Antimony smart editor, AMAS (Automated Model Annotation System)</td></tr><tr><td style=text-align:left>Ron Henkel</td><td style=text-align:left>Medical Informatics Laboratory, Institute for Community Medicine, University Medicine Greifswald</td><td style=text-align:left>In person<br>CellML, COMBINE Archive, SBGN, SBML<br>FAIR</td></tr><tr><td style=text-align:left>Adel Heydarabadipour</td><td style=text-align:left>University of Washington</td><td style=text-align:left>In person<br>Multicellular modeling, SBGN, SBML, SBOL and SBOL Visual</td></tr><tr><td style=text-align:left>Yan-kay Ho</td><td style=text-align:left>University of Cambridge</td><td style=text-align:left>Remotely (virtually)<br>SBML, SBOL and SBOL Visual<br><a href=https://openbioeconomy.org/research_categories/toolkits/>https://openbioeconomy.org/research_categories/toolkits/</a> <a href=https://github.com/openplant/openplant_automation_protocols>https://github.com/openplant/openplant_automation_protocols</a></td></tr><tr><td style=text-align:left>Diego Jahn</td><td style=text-align:left>Center for Information Services and High Performance Computing (ZIH), TU Dresden, Germany</td><td style=text-align:left>Remotely (virtually)<br>Multicellular modeling, MultiCellML, MorpheusML<br>Morpheus (<a href=https://morpheus.gitlab.io/),>https://morpheus.gitlab.io/),</a> MorpheusML Model Repository (<a href=https://morpheus.gitlab.io/model/>https://morpheus.gitlab.io/model/</a>)</td></tr><tr><td style=text-align:left>Bart Jardine</td><td style=text-align:left>University of Washington</td><td style=text-align:left>In person<br>COMBINE Archive, Multicellular modeling, OMEX Metadata, SBML, SED-ML<br>Chemical network modeling software and support for SBML</td></tr><tr><td style=text-align:left>Paul jonas Jost</td><td style=text-align:left>University of Bonn</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, SBGN, SBML<br>pyPESTO, petab</td></tr><tr><td style=text-align:left>Jonathan Karr</td><td style=text-align:left>Formic Labs</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, OMEX Metadata, SBML, SED-ML<br>BioSimulators, BioSimulations</td></tr><tr><td style=text-align:left>Sarah Keating</td><td style=text-align:left>UCL</td><td style=text-align:left>Remotely (virtually)<br>SBML</td></tr><tr><td style=text-align:left>Carlos Lopez</td><td style=text-align:left>Altos Laboratories</td><td style=text-align:left>Remotely (virtually)<br>Multicellular modeling, SBML, models as computer programs<br>PySB, PyDREAM, PyViPR, all things multiscale modeling</td></tr><tr><td style=text-align:left>Augustin Luna</td><td style=text-align:left>Harvard Medical School</td><td style=text-align:left>Remotely (virtually)<br>BioPAX, OMEX Metadata, SBGN, SBOL and SBOL Visual<br>Pathway Commons (pathwaycommons.org); SBGN (sbgn.org); libsbgn (github.com/sbgn/libsbgn); stonpy (github.com/adrienrougny/stonpy)</td></tr><tr><td style=text-align:left>Karin Lundengård</td><td style=text-align:left>Auckland Bioengineering Institute</td><td style=text-align:left>Remotely (virtually)<br></td></tr><tr><td style=text-align:left>Harvey Mannering</td><td style=text-align:left>UCL</td><td style=text-align:left>In person<br>NeuroML, SBML, SED-ML<br>Tools and standards for single-cell modeling in neuroscience. Currently involved in the ongoing development of libSBML</td></tr><tr><td style=text-align:left>Elebeoba May</td><td style=text-align:left>University of Wisconsin-Madison</td><td style=text-align:left>Remotely (virtually)<br>BioPAX, CellML, COMBINE Archive, Multicellular modeling, SBML, SED-ML<br>My lab develops models at multiple scales ranging from gene network, metabolic network, signal transduction to multicellular models. We model individual cells (bacteria and mammalian) and interactions between cells (microbial and host-microbe).</td></tr><tr><td style=text-align:left>Gerhard Mayer</td><td style=text-align:left>HITS gGmbH</td><td style=text-align:left>Remotely (virtually)<br>BioPAX, CellML, COMBINE Archive, Multicellular modeling, NeuroML, OMEX Metadata, SBGN, SBML, SBOL and SBOL Visual, SED-ML, identifiers.org<br>EDITH (European Virtual Human Twin project)</td></tr><tr><td style=text-align:left>Pedro Mendes</td><td style=text-align:left>UConn Health</td><td style=text-align:left>Remotely (virtually)<br>OMEX Metadata, SBML, SED-ML<br>COPASI</td></tr><tr><td style=text-align:left>Blinov Michael</td><td style=text-align:left>UConn Health</td><td style=text-align:left>Remotely (virtually)<br>BioPAX, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBGN, SBML, SED-ML<br>VCell, BioNetGen, SBML</td></tr><tr><td style=text-align:left>Tom Mitchell</td><td style=text-align:left>Raytheon BBN Technologies</td><td style=text-align:left>Remotely (virtually)<br>SBOL and SBOL Visual<br>pySBOL3, SBOL-utilities</td></tr><tr><td style=text-align:left>Ion Moraru</td><td style=text-align:left>UConn Health</td><td style=text-align:left>In person<br>COMBINE Archive, Multicellular modeling, SBML, SED-ML<br><a href=http://vcell.org/,>http://vcell.org/,</a> <a href=http://biosimulations.org/>http://biosimulations.org/</a></td></tr><tr><td style=text-align:left>Sébastien Moretti</td><td style=text-align:left>SIB Swiss Institute of Bioinformatics</td><td style=text-align:left>Remotely (virtually)<br>SBML<br><a href=https://www.metanetx.org/>https://www.metanetx.org/</a></td></tr><tr><td style=text-align:left>Robert Mueller</td><td style=text-align:left>TU Dresden</td><td style=text-align:left>Remotely (virtually)<br>Multicellular modeling, Morpheus</td></tr><tr><td style=text-align:left>Chris Myers</td><td style=text-align:left>University of Colorado Boulder</td><td style=text-align:left>In person<br>COMBINE Archive, SBML, SBOL and SBOL Visual, SED-ML<br>SynBioHub, SynBioSuite, iBioSim</td></tr><tr><td style=text-align:left>David Nickerson</td><td style=text-align:left>Auckland Bioengineering Institute, University of Auckland</td><td style=text-align:left>In person<br>CellML, COMBINE Archive, OMEX Metadata, SED-ML<br>Physiome Model Repository; Center for Reproducible Biomedical Modeling; Physiome journal</td></tr><tr><td style=text-align:left>Alexander Patrie</td><td style=text-align:left>Uconn Health</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, OMEX Metadata, SBML, SED-ML<br>Biosimulators: <a href=https://www.biosimulators.org>https://www.biosimulators.org</a></td></tr><tr><td style=text-align:left>Tim Rudge</td><td style=text-align:left>newcastle university</td><td style=text-align:left>Remotely (virtually)<br>Multicellular modeling, SBML, SBOL and SBOL Visual, SED-ML<br>Flapjack, LOICA</td></tr><tr><td style=text-align:left>Sai Samineni</td><td style=text-align:left>University of Colorado Boulder</td><td style=text-align:left>Remotely (virtually)<br>SBOL and SBOL Visual, LabOp<br>LabOp, Army Data Standards</td></tr><tr><td style=text-align:left>Jim Schaff</td><td style=text-align:left>UConn Health</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, Multicellular modeling, OMEX Metadata, SBML, SED-ML<br>SBML Spatial</td></tr><tr><td style=text-align:left>T.j. Sego</td><td style=text-align:left>University of Florida</td><td style=text-align:left>In person<br>CellML, COMBINE Archive, Multicellular modeling, SBGN, SBML, SED-ML<br>CompuCell3D, Tellurium, Tissue Forge</td></tr><tr><td style=text-align:left>Bilal Shaikh</td><td style=text-align:left>DeepOrigin</td><td style=text-align:left>Remotely (virtually)<br>CellML, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBML, SED-ML<br>Biosimulations</td></tr><tr><td style=text-align:left>Rahuman Sheriff</td><td style=text-align:left>EMBL-EBI</td><td style=text-align:left>Remotely (virtually)<br>CellML, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBGN, SBML, SED-ML, FROG<br>SBML, BioModels, FROG, FAIRSPACE</td></tr><tr><td style=text-align:left>Woosub Shin</td><td style=text-align:left>University of Auckland</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, OMEX Metadata, SBML</td></tr><tr><td style=text-align:left>Lucian Smith</td><td style=text-align:left>University of Washington</td><td style=text-align:left>In person<br>CellML, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBML, SED-ML<br>Antimony, Tellurium</td></tr><tr><td style=text-align:left>Hugh Sorby</td><td style=text-align:left>Auckland Bioengineering Institute</td><td style=text-align:left>In person<br>CellML, COMBINE Archive, SED-ML</td></tr><tr><td style=text-align:left>Ryan Spangler</td><td style=text-align:left>Altos Labs</td><td style=text-align:left>In person<br>Multicellular modeling<br>Vivarium</td></tr><tr><td style=text-align:left>Jörn Starruß</td><td style=text-align:left>Technische Universität Dresden</td><td style=text-align:left>Remotely (virtually)<br>COMBINE Archive, Multicellular modeling, SBML, SED-ML</td></tr><tr><td style=text-align:left>Nguyen Tung</td><td style=text-align:left>EMBL-EBI</td><td style=text-align:left>Remotely (virtually)<br>BioPAX, COMBINE Archive, Multicellular modeling, OMEX Metadata, SBGN, SBML, SED-ML<br>BioModels, COMBINE Archive 2.0</td></tr><tr><td style=text-align:left>Lorenzo Veschini</td><td style=text-align:left>King&rsquo;s Collge London</td><td style=text-align:left>Remotely (virtually)<br>Multicellular modeling, OMEX Metadata, SBML</td></tr><tr><td style=text-align:left>Gonzalo andrés Vidal peña</td><td style=text-align:left>Newcastle University</td><td style=text-align:left>Remotely (virtually)<br>SBOL and SBOL Visual<br><a href=https://github.com/RudgeLab/PUDU>https://github.com/RudgeLab/PUDU</a> <a href=https://github.com/RudgeLab/LOICA>https://github.com/RudgeLab/LOICA</a></td></tr><tr><td style=text-align:left>Carolus Vitalis</td><td style=text-align:left>University of Colorado Boulder</td><td style=text-align:left>Remotely (virtually)<br>SBOL and SBOL Visual</td></tr><tr><td style=text-align:left>Khoi Vo</td><td style=text-align:left>University of California, Riverside</td><td style=text-align:left>In person<br>BioPAX, CellML, COMBINE Archive, Multicellular modeling, NeuroML, OMEX Metadata, SBGN, SBML, SBOL and SBOL Visual, SED-ML<br>Multiscale modeling of blood clot</td></tr><tr><td style=text-align:left>Daniel Weindl</td><td style=text-align:left>Helmholtz Munich</td><td style=text-align:left>Remotely (virtually)<br>SBML, SED-ML, PEtab<br><a href=https://github.com/AMICI-dev/AMICI/>https://github.com/AMICI-dev/AMICI/</a> <a href=https://github.com/ICB-DCM/pyPESTO/>https://github.com/ICB-DCM/pyPESTO/</a> <a href=https://petab.readthedocs.io/en/latest/>https://petab.readthedocs.io/en/latest/</a></td></tr><tr><td style=text-align:left>Felipe Xavier buson</td><td style=text-align:left>Cambridge University</td><td style=text-align:left>Remotely (virtually)<br>SBOL and SBOL Visual</td></tr><tr><td style=text-align:left>Guillermo Yanez feliu</td><td style=text-align:left>Newcastle University</td><td style=text-align:left>Remotely (virtually)<br>Multicellular modeling, Flapjack<br>Multicellular pattern formation, computational synthetic biology, microscopy, microfluidics and image analysis.</td></tr><tr><td style=text-align:left>Fengkai Zhang</td><td style=text-align:left>NIAID/NIH</td><td style=text-align:left>In person<br>COMBINE Archive, Multicellular modeling, SBGN, SBML, SED-ML<br>Simmune</td></tr></tbody></table><h2><a href=https://forms.gle/Nqec2JdyRj12HobK9 target=_blank rel=noopener><span style=color:#8dc742>Registration</span> <i class="fas fa-chevron-right" aria-hidden=true></i></a></h2>Registration for the meeting is open and is free. Please register at the link above as soon as possible. This will help us plan the schedule and match your interests to the timing of the breakouts, etc. Note, only registered attendees will be sent information related to video conferencing links, etc.<h2><a href=https://forms.gle/rYZhqbekKLs1RY3B7 target=_blank rel=noopener><span style=color:#8dc742>Call for Breakout Sessions</span> <i class="fas fa-chevron-right" aria-hidden=true></i></a></h2>All attendees can suggest breakout sessions for hacking and/or detailed discussions of certain aspects of one or several of the COMBINE standard(s), metadata and semantic annotations (format-specific or overarching), application and implementations of the COMBINE standards, or any other topic relevant for the COMBINE community. The topics for those breakout sessions, and the time slots which would suit their communities can be submitted via the link above. Note, breakout session organisers will be responsible for creating and hosting their own online sessions, if required.<h2><a href=https://forms.gle/hcpWJrKCtFQDuPwR6 target=_blank rel=noopener><span style=color:#8dc742>Call for Lightning Talks and Posters</span> <i class="fas fa-chevron-right" aria-hidden=true></i></a></h2>Requests for a lightning talk (3 min max.) and/or poster can be submitted via the link above. Please use several forms if you want to submit abstracts on different topics. The submission deadline is outlined above. Talks will take place during the community session and posters will be displayed throughout the meeting.<h2 style=color:#8dc742>Topics of Interest</h2><ul><li>Data exchange, pipelines and model standards for systems and synthetic biology</li><li>Visualization and graphical notation standards for systems and synthetic biology</li><li>Standards for sharing and analysing biological pathway data</li><li>Standards for computational biological models and modelling support</li><li>Metadata description and model annotation in COMBINE standard formats</li><li>Implementation of COMBINE standards in tools, databases and other resources</li><li>Integrated model and data management for systems and synthetic biology</li><li>Standardization of Artificial Intelligence approaches in biological modelling</li><li>Emerging standardization needs and multicellular modeling</li><li>Community aspects of COMBINE</li></ul></div></div></div></section><section id=future_events class="home-section wg-people"><div class=container><div class="row justify-content-center people-widget"><div class=col-md-12><h2 class=mb-4>Future Events</h2></div><div class="col-12 col-sm-auto people-person"><div class=portrait-title><h2><a href=/author/combine-2023/>COMBINE 2023</a></h2></div></div></div></div></section><section id=associated-standards class="home-section wg-people"><div class=container><div class="row justify-content-center people-widget"><div class=col-md-12><h2 class=mb-4>Past Events</h2></div><div class="col-12 col-sm-auto people-person"><div class=portrait-title><h2><a href=/author/combine-2021/>COMBINE 2021</a></h2></div></div><div class="col-12 col-sm-auto people-person"><div class=portrait-title><h2><a href=/author/combine-2022/>COMBINE 2022</a></h2></div></div><div class="col-12 col-sm-auto people-person"><div class=portrait-title><h2><a href=/author/harmony-2021/>Harmony 2021</a></h2></div></div><div class="col-12 col-sm-auto people-person"><div class=portrait-title><h2><a href=/author/harmony-2022/>Harmony 2022</a></h2></div></div><div class="col-12 col-sm-auto people-person"><div class=portrait-title><h2><a href=/author/organizing-a-combine-event/>Organizing a COMBINE event</a></h2></div></div></div></div></section><section id=past_events class="home-section wg-blank" style=background-color:#fff><div class=container><div class=row><div class=col-lg-12><div style=text-align:center><p><a href=http://old_co.mbine.org/events target=_blank rel=noopener>Events prior to 2021</a></p></div></div></div></div></section><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script src=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.js integrity="sha512-SeiQaaDh73yrb56sTW/RgVdi/mMqNeM2oBwubFHagc5BkixSpP1fvqF47mKzPGWYSSy4RwbBunrJBQ4Co8fRWA==" crossorigin=anonymous></script><script>const code_highlighting=true;</script><script>const search_config={"indexURI":"/index.json","minLength":1,"threshold":0.3};const i18n={"no_results":"No results found","placeholder":"Search...","results":"results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'talk':"Talks",'slides':"Slides"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/js/wowchemy.min.4c2bca31150ce93c5a5e43b8a50f22fd.js></script><div class=container><footer class=site-footer><p class=powered-by></p><p class=powered-by>Published with
<a href=https://wowchemy.com target=_blank rel=noopener>Wowchemy</a> —
the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.
<span class=float-right aria-hidden=true><a href=# class=back-to-top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div></body></html>